<CENTER><H1>BASIC MDS INVESTIGATIONS</H1></CENTER>

<A href="https://notebooks.antigenic-cartography.org/dsmith/mds/index.html">Derek's page</A> at antigenic-cartography.org (and <A href="/home/dsmith/mds/src/mds/mds/index.html">local</A>)
<br>
<A href="https://notebooks.antigenic-cartography.org/fouchier/mds/index.html">Ron's page</A> at antigenic-cartography.org
<br>
<A href="https://www.santafe.edu/~asl/mds/index.html">Alan's page</A> at SFI (link broken)
<br>
<A href="https://notebooks.antigenic-cartography.org/colin/mds/index.html">Colin's page</A> at antigenic-cartography.org
<br>
<A href="https://notebooks.antigenic-cartography.org/robfarrow/mds/index.html">Rob's page</A> at antigenic-cartography.org (and <A href="/home/dsmith/mds/src/mds/mds/rob-index.html">local</A>)
<br>
<A href="https://notebooks.antigenic-cartography.org/bjorn/index.html">Bjorn's page</A> at antigenic-cartography.org
<br>
<A href="https://notebooks.antigenic-cartography.org/dan/mds/index.html">Dan's page</A> at antigenic-cartography.org
<br>
<A href="https://notebooks.antigenic-cartography.org/nicola/">Nicola's page</A> at antigenic-cartography.org
<br>
<A href="https://notebooks.antigenic-cartography.org/eu/index-full.html">Eu's page</A> at antigenic-cartography.org
<br>
<A href="https://notebooks.antigenic-cartography.org/ana/">Ana's page</A> at antigenic-cartography.org
<br>
<A href="https://notebooks.antigenic-cartography.org/dave/mds/">David's page</A> at antigenic-cartography.org
<br>
<A href="https://notebooks.antigenic-cartography.org/margaret/">Margaret's page</A> at antigenic-cartography.org
<br>
<A href="https://data.antigenic-cartography.org/yan/mds/investigations/landscapes/ferret_some/index.html">One of Yan's pages (reset to index when ac-org is back)</A> at antigenic-cartography.org
<br>
<A href="https://notebooks.antigenic-cartography.org/miranda/index.html">Miranda's page</A> at antigenic-cartography.org
<br>
<A href="https://notebooks.antigenic-cartography.org/ramona/index.html">Ramona's page</A> at antigenic-cartography.org
<br>
<A href="https://notebooks.antigenic-cartography.org/slj38/">Sarah's page</A> at antigenic-cartography.org
<BR>
<A href="https://notebooks.antigenic-cartography.org/mario/">Mario's page</A> at antigenic-cartography.org
<BR>
<A href="https://notebooks.antigenic-cartography.org/judy/">Judy's's page</A> at antigenic-cartography.org
<BR>
<A href="https://notebooks.antigenic-cartography.org/samwilks/">Sam's page</A> at antigenic-cartography.org
<BR>
<A href="https://notebooks.antigenic-cartography.org/hr283/mds/">Hannah's page</A> at antigenic-cartography.org
<BR>
<A href="https://notebooks.antigenic-cartography.org/lck33/index.html">Leah's page</A> at antigenic-cartography.org
<BR>
<A href="https://notebooks.antigenic-cartography.org/terry/index.html">Terry's page</A> at antigenic-cartography.org
<BR>
<A href="https://notebooks.antigenic-cartography.org/barbara/index.html">Barbara's page</A> at antigenic-cartography.org


<BR>
<br>
<A href="https://notebooks.antigenic-cartography.org/cc-data/index.html">VCM data (tables only)</A>
<BR>
<A href="https://notebooks.antigenic-cartography.org/who/whocc-tables.html">VCM data (tables + routine diagnostics + timeseries)</A>
<BR>
<A href="https://notebooks.antigenic-cartography.org/eu/cc-tables.html">Our VCM data (tables + routine diagnostics + timeseries)</A>
<BR>
<A href="https://notebooks.antigenic-cartography.org/who/whocc-2009-h1n1.html">H1N1pdm data (tables + routine diagnostics)</A>
<BR>
<A href="http://notebooks.antigenic-cartography.org/H1N1-mexican-2009/">H1N1pdm public</A>
<BR>
<A href="https://notebooks.antigenic-cartography.org/who/whocc-b-vic.html">B Vic (tables + routine diagnostics)</A>
<BR>
<A href="https://notebooks.antigenic-cartography.org/who/whocc-b-yam.html">B Yam (tables + routine diagnostics)</A>
<BR>
<A href="https://notebooks.antigenic-cartography.org/cc-data/full-index.html">All imported tables</A>
<BR>




<H1> Panel distance </H1>
  <UL>
  <li><a href="investigations/panel-distance/panel-distance.html">Pannel distance</a>
  <li><a href="investigations/panel-distance/different-num-sera-normalization.html">Test removing some sera</a>
  </UL>

<H1> Ordinal/Metric comparison </H1>
  <UL>
  <li><a href="investigations/ordinal-metric/95-96-a.html">95-96-a table</a>
  <li><a href="investigations/ordinal-metric/94-95-7.html">94-95-7 table</a>
  <li><a href="investigations/ordinal-metric/common-antigens.html">Common antigens</a>
  <li><a href="investigations/ordinal-metric/tab5.html">Table 5 (vaccine strains)</a>
  <li><a href="investigations/ordinal-metric/tab23.html">Table 23</a>
  <p>
  <li><a href="investigations/mock-hi-tables/mock-hi-tables.html">Mock HI tables</a>
  <li><a href="investigations/mock-hi-tables/mock-hi-tables-2-fold-radius.html">Mock HI tables (2-fold radius)</a>
  </UL>

<H1> Perturbing HI tables</H1>
  <UL>
  <li><a href="investigations/piecing-hi-tables/by-big-merge/perturbing-hi-tables.html">Perturbing HI tables to see how well MDS can repair them</a>
  <li>Lots of stuff in <a href="investigations/mock-hi-tables/mock-hi-tables.html">Mock HI tables</a>
  <li>Lots of stuff in <a href="investigations/mock-hi-tables/mock-hi-tables-2-fold-radius.html">Mock HI tables (2-fold radius)</a>
  </UL>

<H1> Using correlation instead of euclidean distance</H1>
  <UL>
  <li>Some stuff at the bottom of <a href="investigations/1999-and-2000/1999-and-2000.html">Look at 1999 and 2000 tables</a>
  <li>Some stuff close to the bottom of <a href="investigations/panel-distance/panel-distance.html">Pannel distance</a>
  <li>Some stuff 2/3rds the way down <a href="investigations/piecing-hi-tables/by-big-merge/vaccine-strains-in-individual-tables.html">Vaccine strains in individual tables</a>
  </UL>

<H1> Using Manhattan instead of euclidean distance</H1>
  <UL>
  <li>Some stuff at the bottom of <a href="investigations/mock-hi-tables/mock-hi-tables-2-fold-radius.html">Mock HI tables (2-fold radius)</a>
  </UL>

<H1> Tendency to curl</H1>
  <UL>
  <li><a href="investigations/tendency-to-curl/tendency-to-curl.html">Tendency to curl with diagonal distance matrix (demonstration on simulated data)</a>
  <li><a href="investigations/tendency-to-curl/tendency-to-curl-with-full-distance-matrix.html">Tendency to curl with full distance matrix (not as dont-cares) (demonstration on simulated data)</a>
  <li><a href="investigations/tendency-to-curl/add-mds-distance-to-stress-on-dont-care.html">Add MDS distance to stress on dont-care (trying to reduce curl)</a>
  <li><a href="investigations/tendency-to-curl/stretch-endpoints.html">Stretch endpoints (trying to reduce curl)</a>
  </UL>

<H1> Singular Value Decomposition (SVD)</H1>
  <UL>
  <li><a href="investigations/svd/svd.html">Test of SVD and MDS on the concentric-circle simulated data</a>
  </UL>

<H1> HI Metric </H1>
  <UL>
  <li><a href="investigations/hi-metric/asl/Dec2000_BestMerged/main.html">On Dec 2000, "best merged" (ASL)</a>
  <li><a href="investigations/hi-metric/main.html">Initial experiments</a>
  <li>(<a href="investigations/hi-metric/concentric-circle-test-cases.html">Concentric circle test cases</a>)
  <li>(<a href="investigations/hi-metric/code-mods.html">Mods to the production code to do HI metric (and also row and col adjsuts)</a>)
  <li><a href="investigations/hi-metric/lt40-tests.html">Test mixed <40s and <10s</a>
  <p>
  <li><a href="investigations/hi-metric/slopes.html">Slopes</a>
  <li><a href="investigations/hi-metric/slopes-77-87.html">Slopes -- use on the 77-87 merge</a>
  <li><a href="investigations/hi-metric/slopes-mid90s.html">Slopes -- use on the mid90s merge</a>
  <p>
  <li><a href="investigations/hi-metric/titer-distributions.html">Titer distributions (for the sequenced-strains merge take 4)</a>
  <li><a href="investigations/hi-metric/concentric-circle-test-cases-colinear-sera.html">Concentric circle test cases on colinear sera</a>
  <li><a href="investigations/hi-metric/concentric-circle-test-cases-colinear-sera-col-adjust.html">Concentric circle test cases on colinear sera -- col adjusts</a>
  <li><a href="investigations/hi-metric/colinear-fake-alan.html">Linear evolution fake data (from asl)</a>
  <li><a href="investigations/hi-metric/colinear-fake-alan-checking.html">Linear evolution fake data -- running square subsets</a>
  <li><a href="investigations/dimensional-annealing/gradual-anneal.html">Smoother annealing schedule on the linear evolution fake data (also listed under dim annealing section below)</a>
  <p>
  <li><h3>Experiments with Anika and Kai (January 2010)</h3>  
  <li><a href="investigations/hi-metric/anika-kai-concentric-circle-test-case.html">Initial concentric-circle test case (January 2010)</a>
  </UL>



<H1> Avidity and sera reactivity variation (with HI Metric) </H1>
  <UL>
  <li><a href="investigations/hi-metric/col-adjust.html">Variation in sera</a>
         (<a href="investigations/hi-metric/aside-different-lengths-of-p-vector.html">aside: different length vectors in conjugant gradient</a>)
         (<a href="investigations/hi-metric/aside-2x-difference-in-coordss.html">aside: different input and output coordss</a>)
  <li><a href="investigations/hi-metric/row-col-adjust.html">Variation in avidity and sera (on concentric circle data)</a>
  <li><a href="investigations/hi-metric/row-col-adjust-real-tables.html">Variation in avidity and sera (on real tables)</a>
  <li>(<a href="investigations/hi-metric/concentric-circle-test-cases.html">Concentric circle test cases</a>)
  <li>(<a href="investigations/hi-metric/code-mods.html">Mods to the production code to do HI metric (and also row and col adjsuts)</a>)
  <li><a href="investigations/hi-metric/squirting-out.html">Strains squirting out</a>
  <li><a href="investigations/hi-metric/error-lines.html">Feeling out and checking the error-line tool</a>  
  <p>
  <li><a href="investigations/row-col-adjusts/hard-code-row-col-adjusts.html">Row/col adjusts take 2</a>  
  <li><a href="investigations/row-col-adjusts/penalty-for-row-col-adjust.html">Penalty for Row/col adjusts</a>  
  <p>
  <li><a href="investigations/row-col-adjusts/one-adjust-at-a-time.html">One adjust at a time (March 2002, take 3)</a>  
  <p>
  <li><a href="investigations/merge-hi-tables/sera-in-3d.html">Allow the sera to be in 3d, restrict the antigens to 2d, adjust the col bases (link is also in the shape section below)</a>
  <li><a href="investigations/merge-hi-tables/seq-t6-col-adjust.html">Col adjusts on seq-t6 after non-adjust initial position (November 2002)</a>  
  <li><a href="investigations/row-col-adjusts/adjusts-with-fixed-coordss.html">Adjusts with fixed coordss (November 2002)</a>  
  </UL>

<H1> Less than 10 (with HI metric) (MOVE THE MERGE STUFF TO THE SHAPE SECTION BELOW)</H1>
  <UL>
  <li><a href="investigations/less-than-ten/initial-experiments.html">Initial experiments</a>
  <li><a href="investigations/less-than-ten/initial-merged-hi-table-runs.html">First runs on the merged HI table</a> (<a href="investigations/less-than-ten/setup.html">setting up the HI tables</a>)
  <li><a href="investigations/less-than-ten/merged-hi-table-experiments.html">Early merged HI table experiments</a>
  <li><a href="investigations/less-than-ten/error-lines.html">Examining the early merged HI table results</a> (and introducing the "error line" tool)
  <li><a href="investigations/less-than-ten/comparison-with-sequence.html">Comparison with sequence</a>
  <li><a href="investigations/less-than-ten/fewer-lt10s.html">Fewer less than 10's</a>
  <li><a href="investigations/less-than-ten/more-dimensions.html">3 4 and 5 dimension runs</a>
  <li><a href="investigations/less-than-ten/initial-hacked-avidity-reactivity-adjusts.html">Initial hacks on adjusting avidity and reactivity</a>
  <li><a href="investigations/less-than-ten/avidity-reactivity-adjusts.html">Adjusting avidity and reactivity</a>
  <li><a href="investigations/less-than-ten/avidity-reactivity-adjusts-take-2.html">Adjusting avidity and reactivity--Take 2</a>
  <li><a href="investigations/less-than-ten/less-than-fourty.html">Less than 40</a>
  </UL>

<H1> Dimensional Annealing (add the stuff from alan)</H1>
  <UL>
  <li><a href="investigations/dimensional-annealing/concentric-circle.html">On 2d perfect concentric circle</a>
  <li><a href="investigations/dimensional-annealing/schedules.html">Test different schedules</a>
  <li><a href="investigations/dimensional-annealing/alan-seq-mds/sync-test.html">Syncing sequence dim anneal</a>
  <li><a href="investigations/dimensional-annealing/gradual-anneal.html">Smoother annealing schedule (also listed under HI metric above)</a>
  </UL>

<H1> Antibody fitness landscapes</H1>
  <UL>
  <li><a href="investigations/landscapes/index.html">Examples of landscapes</a>
  <li><a href="investigations/landscapes/landscapes-with-titers.html">Vaccine trial sample pre, post, and delta landscapes, with titers</a>
  <li><a href="investigations/landscapes/data-for-stephen-and-yan.html">Human and ferret data for Stephen and Yan</a>
  <li><a href="investigations/landscapes/data-for-yan-2006-07-13.html">Human and ferret data for Yan (2006-07-13)</a>
  </UL>

<H1> Misc investigations</H1>
  <UL>
  <li><a href="investigations/avidity/looking-for-low-avidity.html">Looking for low avidity strains</a>
  <li><a href="investigations/low-titers/thoughts.html">Thoughts about low titers</a>
  <li><a href="investigations/basis-measurement/initial-thoughts.html">Thoughts about measuring the "basis" for a point</a>
  <li><a href="investigations/yam-vic/landscape.html">B/Vic B/Yam vaccine strain selection strategies -- comparison of current with Ana's decision-tree strategy (2010-12-15)</a>
  <li><a href="investigations/hcv/han.html">HCV with Han (2012-03-22)</a>
  <li><a href="investigations/misc/b-vic-yam-cluster-transitions.html">Find sequences that have fewest substitutions other than a particular substitution (2013-02-14)</a>
  <li><a href="investigations/misc/b-vic-yam-cluster-transitions-vic-to-hi.html">Melb B/Vic viruses to HI for Malysia to Brisbane cluster transisition (2013-02-21)</a>
  <li><a href="investigations/misc/b-vic-yam-cluster-transitions-yam-to-hi.html">Melb B/Yam viruses to HI for Malysia to Brisbane cluster transisition (2013-02-21)</a>
  <p>
  <li><a href="investigations/misc/merge-2004-and-oz2013.html">Merge 2004 and Oz 2013 (with Judy) (2013-08-02)</a>
  <p>
  <li><a href="investigations/h2/index.html">H2 regression (Martin/Ron) (2015-02-24)</a>
  </UL>



<HR>

<CENTER><H1>APPLICATION OF MDS</H1></CENTER>

<H1> Jan tables </H1>
  <UL>
  <li><a href="investigations/piecing-hi-tables/start-looking-at-jan-tables.html">Feeling out the individual Jan tables</a>
  <li><a href="investigations/piecing-hi-tables/table-ags-in-sequence-mds/main.html">Each new HI table shown on the sequence map</a>
  <li><a href="investigations/piecing-hi-tables/table-ags-in-sequence-mds/main2.html">Overlaps between the new HI tables shown on the sequence map</a>

  <li><a href="investigations/piecing-hi-tables/start-merging-jan-hi-tables.html">Initial HI merge experiments of Jan tables</a>
  <p>
  <li><a href="investigations/piecing-hi-tables/by-big-merge/vaccine-strains.html">Vaccine strains</a>
  <li><a href="investigations/piecing-hi-tables/by-big-merge/vaccine-strains-in-individual-tables.html">Vaccine strains in individual tables</a>
  </UL>

<H1> Joining HI tables by merging of panel distance matrices</H1>
  <UL>
  <li><a href="investigations/piecing-hi-tables/original-tables-merge-distance-matrices.html">Analysis of the original tables and merging by distance matrix</a>
  </UL>

<H1> Joining tables merging of HI data</H1>
  <UL>
  <li><a href="investigations/piecing-hi-tables/by-big-merge/best-merge-dec-2000.html">Best merge (as of Dec 2000)</a>
  <p>
  <li><a href="investigations/piecing-hi-tables/by-big-merge/by-big-merge.html">Workings to set up the big merge</a>
  <li><a href="investigations/piecing-hi-tables/by-big-merge/mds-merged-hi.html">Further workings to set up merge, and pannel-metric MDS of the merged table</a>
  <li><a href="investigations/asl-merged-hi-tables/main.html">MDS of the big merged HI table (ASL) (ordinal?)</a>
  <p>
  <li><h3>Take 2 (from August 2001)</h3>
  <li><a href="investigations/merge-hi-tables/verbose-merge.html">Verbose merge</a>
  <li><a href="investigations/merge-hi-tables/2001-09-merge.html">2001-09  Merge 67 tables all at once</a>
  <li><a href="investigations/merge-hi-tables/2001-10-merge.html">2001-10  Merge 67 tables, stepwise changes from old merge</a>
  <li><a href="investigations/merge-hi-tables/2001-11-merge.html">2001-11  Merge 67 tables, continue to stepwise</a>
  <li><a href="investigations/merge-hi-tables/te77-gap-investigation.html">Te77 gap investigation (and tab68 integration)</a>
  <li><a href="investigations/merge-hi-tables/2002-01-merge.html">2002-01  Tab 68 and Vaccine table added into 2001-11 merge</a>
  <li><a href="investigations/merge-hi-tables/2002-01-merge-summary.html">2002-01  Tab 68 and Vaccine table added into 2001-11 merge (summary of above)</a>
  <li><a href="investigations/merge-hi-tables/2002-01-merge-vaccine-table-prediction.html">2002-01  Vaccine table prediction</a>
  <li><a href="investigations/merge-hi-tables/2002-02-name-updates.html">2002-02  Curation of data, some name changes as result of ron's collection of isolation dates</a>
  <li><a href="investigations/merge-hi-tables/2002-02-merge-vaccine-table-prediction.html">2002-02  Vaccine table prediction</a>
  <li><a href="investigations/merge-hi-tables/te77-gap-investigation-take-2.html">Te77 gap investigation (Take 2 -- feb2002)</a>
  <li><a href="investigations/merge-hi-tables/tables-4-6-7-8.html">Tables 4 6 7 8 into the merge?</a>
  <li><a href="investigations/merge-hi-tables/2002-02-continue-carefully.html">2002-02  continue carefully</a>
  <li><a href="investigations/merge-hi-tables/the-ba79-clade.html">The BA/79 clade</a>
  <p>
  <li><a href="investigations/merge-hi-tables/75-85-by-single-tables.html">The 75 to 85 period by merging individual tables one at a time</a>
  <li><a href="investigations/merge-hi-tables/tabs-11-and-48.html">Focus on tabs 11 and 48 out of the tabs in the 75 to 85 period</a>
  <li><a href="investigations/merge-hi-tables/tabs-11-48-12-8.html">Proceed from the previous merge of tables 11 and 48, and add the similar tables 12 and 8</a>
  <li><a href="investigations/merge-hi-tables/add-in-tab10-to-the-11-48-12-8-merge.html">Add in table 10 to the tables 11 48 12 and 8 merge (10 has more sera, the other tables use approx the same sera)</a>
  <li><a href="investigations/merge-hi-tables/what-to-do-with-tables-9-and-68.html">What to do with tables 9 and 68</a>
  <li><a href="investigations/merge-hi-tables/80s-hi-merge-summary.html">1980s HI table merge summary</a>
  <li><a href="investigations/merge-hi-tables/checking-the-right-middle-clade.html">Checking the MDS clade structure with the HI data</a>
  <li><a href="investigations/merge-hi-tables/prototypes.html">Compare Jan's monoclonal-identified-prototypes with the MDS plot</a>
  <li><a href="investigations/merge-hi-tables/sera-landscapes.html">Sera landscapes for the 1980s merge</a>
  <li><a href="investigations/merge-hi-tables/try-adding-back-te1a77-sr.html">Try adding back TE/1a/77-sr</a>
  <li><a href="investigations/merge-hi-tables/80s-sequence.html">Compare with the sequences for the 80s</a>
  <p>
  <li><a href="investigations/merge-hi-tables/table-range.html">Table ranges -- the years of the antigens and sera</a>
  <li><a href="investigations/merge-hi-tables/70s-hi.html">Merge 70s (we know this well, tables 1 2 and 3)</a>
  <li><a href="investigations/merge-hi-tables/merge-late-80s-early-90s.html">Merge late 80s and early 90s</a>
  <li><a href="investigations/merge-hi-tables/merge-late-80s-early-90s-sequenced-strains.html">Merge late 80s and early 90s - only use sequenced or prototpye strains for later tables</a>
  <li><a href="investigations/merge-hi-tables/merge-late70s-to-mid90s-sequenced-strains.html">Merge 77 to 95 - only use sequenced or prototpye strains for later tables</a>
  <li><a href="investigations/merge-hi-tables/merge-77-to-97.html">Merge 77 to 97 - sequenced and protyptype only (except all antigens prior to 87)</a>
  <li><a href="investigations/merge-hi-tables/merge-77-to-97-add-lt10s.html">Merge 77 to 97 - sequenced and protyptype only (except all antigens prior to 87) -- we see folding so add add <10s</a>
  <li><a href="investigations/merge-hi-tables/merge-mid90s.html">Merge mid 90s, sequenced and protyptype only, we we have now traced the flip into here.</a>
  <li><a href="investigations/merge-hi-tables/sera-in-more-dimensions.html">Merge mid 90s -- try letting the sera be in more dimensions.</a>
  <li><a href="investigations/merge-hi-tables/merge-mid90s-lt40s.html">Merge mid 90s with <40s as <40s not as dont-cares.</a>
  <li><a href="investigations/merge-hi-tables/sydney.html">Sydney -- why do we have banding and dispersion of the Sydney clade?</a>
  <li><a href="investigations/merge-hi-tables/sydney-avidity.html">Sydney -- deal with high avidity strains</a>
  <p>
  <li><a href="investigations/merge-hi-tables/join-pre-post-77.html">Join pre post 77 with table 6.</a>
  <li><a href="investigations/merge-hi-tables/77-link-with-tab72.html">Link pre and post 77 with the newly rediscovered tab72</a>
  <p>
  <li><a href="investigations/merge-hi-tables/curacao-take-3.html">Curacao take 3 -- with dim anneal</a>
  <li><a href="investigations/merge-hi-tables/sequenced-strains-merge-take-4.html">Sequenced strains merge take 4 -- prototypes tables, and trying high and low col adjusts</a>
  <p>
  <li><h3>Check portions of the sequenced strain merge take 4</h3>
  <li><a href="investigations/merge-hi-tables/pre77.html">Pre 77 checking (distorted in take 4, and turned so the join is thru vi75 it seems)</a>
  <li><a href="investigations/merge-hi-tables/77-join.html">77 join</a>
  <li><a href="investigations/merge-hi-tables/post77.html">Post 77 checking</a>
  <li><a href="investigations/merge-hi-tables/dead-end.html">Dead end checking (we are seeing some hemisphering in the sequenced strain merge take 4)</a>
  <li><a href="investigations/merge-hi-tables/jo-wu-data.html">Jo Wu extract for making new table</a>
  <li><a href="investigations/merge-hi-tables/rons-vaccine-table-merge.html">Ron's vaccine table merge take 2 (revisit with dimensional annealing -- have I done this already?)</a>
  <p>
  <li><a href="investigations/merge-hi-tables/rivm-database.html">RIVM database -- are we missing years in our HI tables?</a>
  <p>
  <li><a href="investigations/merge-hi-tables/data-for-alan.html">2001-12  Data for Alan</a>
  <p>
  <li><a href="investigations/viro-tables/merge.html">Viro tables trial merge</a>
  <p>
  <li><a href="investigations/merge-hi-tables/sequenced-strains-merge-take-5.html">Sequenced strains merge take 5 (add in tables 69-74)</a>
  <p>
  <li><a href="investigations/merge-hi-tables/seq-t6-list-merge.html">Seq t6 run with a list of merged values, not the geometric average</a>
  <p>
  <li><h3>The dead end snaking in seq-t7</h3>
  <li><a href="investigations/hi-prediction-from-hi/dead-end-in-seq-t6-jw-an-earlier-look.html">The dead-end snakeing (initial looking around)</a>
  <li><a href="investigations/hi-prediction-from-hi/dead-end-in-seq-t6-jw.html">The dead end snakeing (is the snakeing real -- NO)</a>
  <li><a href="investigations/hi-prediction-from-hi/dead-end-in-seq-t6-jw-part-2.html">The dead end snakeing (what is causing the snaking)</a>
  <li><a href="investigations/hi-prediction-from-hi/dead-end-in-seq-t6-jw-new-data.html">The dead end snakeing (what new measurements to take)</a>
  <li><a href="investigations/merge-hi-tables/dead-end-table.html">Check the new dead end table</a>
  <li><a href="investigations/merge-hi-tables/dead-end-table-part-2.html">More checks on the dead-end table</a>
  <p>
  <li><h3>All strains merge</h3>
  <li><a href="investigations/merge-hi-tables/all-strains-merge.html">All strains merge (take 1, 2002-01)</a>
  <li><a href="investigations/merge-hi-tables/all-strains-merge-take-2.html">All strains merge (take 2, 2002-02)</a>
  <li><a href="investigations/merge-hi-tables/all-strains-merge-2002-06-26.html">All strains merge (take 3, 2002-06)</a>
  <li><a href="investigations/merge-hi-tables/seq-t7.html">All strains merge (take 7 (to match with "seq-t7"), 2002-11) "all-t7"</a>
  <p>
  <li><a href="investigations/merge-hi-tables/tables-69-75.html">The new tables 69-75 with all-strains</a>
  <p>
  <li><a href="investigations/merge-hi-tables/redo-mid-90s.html">Redo HI for mid-90s</a>
  <p>
  <li><h3>Sequenced strains merges</h3>
  <li><a href="investigations/merge-hi-tables/seq-t7.html">Seq t7, 2002-11</a>
  <li><a href="investigations/merge-hi-tables/seq-t8.html">Seq t8, 2003-01</a>
  <li><a href="investigations/merge-hi-tables/seq-t9.html">Seq t9, 2003-03</a>
  <li><a href="investigations/merge-hi-tables/seq-t9a.html">Seq t9a  (Seq t9 without south africa table)</a>
  </UL>

<H1> Shape</H1>
  <UL>
  <p>
  <li><a href="investigations/ui/point-stress-component-tool/main.html">Visualizing point stress and correlation with col basis (also under UI: Point stress component tool)</a>
  <p>
  <li><a href="investigations/procrustes/tab22-fullmerge.html">Tab22 by itself then Procrusted onto the full merge</a>
  <p>
  <li><a href="investigations/shape/num-dimensions.html">Num dimensions (Jan 2002) (ASL)</a>
  <p>
  <li><a href="investigations/shape/all-strains/2d.html">First 2d run on all strains (Jan 2002)</a>
  <li><a href="investigations/shape/all-strains/procrustes-individual-tables-onto-all-strains.html">Procrustes individual tables onto all strains</a>
  <p>
  <li><a href="investigations/shape/all-strains/ForDerek5/README.html">ForDerek5 from ASL -- 5d all strains merge dim annealed with penalty to 2d, pre reoptimization in 2d.</a>
  <li><a href="investigations/shape/all-strains/out-of-plane.html">Investigate the points that want to be out of the 2d plane (on all-strains merge)</a>
  <li><a href="investigations/shape/all-strains/all-strains-merge-take-1-close-points-investigation.html">All strains merge take 1, data for investigation of close points, and for ron's category 1 and 2 antigens</a>
  <li><a href="investigations/shape/all-strains/nl-nc.html">Investigate the NL/NC antigens</a>
  <p>
  <li><h3>Dimensional analysis of the sequenced strains merges t4 and t5</h3>
  <li><a href="investigations/merge-hi-tables/dead-end-in-3d.html">Allow the dead end to optimize in 3d, the rest of the strains in 2d, as a validation</a>
  <li><a href="investigations/merge-hi-tables/early-strains-in-3d.html">Allow the early (pre 86-87 season) strains to optimize in 3d, the rest of the strains in 2d, (lookin for scorpion-tail)</a>
  <li><a href="investigations/merge-hi-tables/seq-t4-and-seq-t5-in-2d.html">Sequenced strain merges (t4 and t5) in 2d</a> (<A href="investigations/merge-hi-tables/seq-t4-and-seq-t5-another-look.html">another look</A>) (<A href="investigations/merge-hi-tables/seq-t4-and-seq-t5-in-2d-add-no-seq-strains.html">try adding non-sequenced strains</A>) (<A href="investigations/merge-hi-tables/strain-crossing-77.html">connection lines of strains crossing 77</A>)
  <li><a href="investigations/merge-hi-tables/seq-t4-and-seq-t5-in-3d.html">Sequenced strain merges (t4 and t5) in 3d: comparison with 2d runs, and comparison with each other</a>
  <li><a href="investigations/merge-hi-tables/seq-t4-and-seq-t5-in-4d.html">Sequenced strain merges (t4 and t5) in 4d</a>
  <li><a href="investigations/merge-hi-tables/seq-t4-and-seq-t5-in-5d.html">Sequenced strain merges (t4 and t5) in 5d: comparison with each other</a>
  <li><a href="investigations/merge-hi-tables/seq-t4-and-seq-t5-dimensions.html">Sequenced strain merges (t4 and t5) dimensional analysis -- summary</a>
  <p>
  <li><a href="investigations/slop/titers-to-measure-2.html">Identifying strains that could most use new titers</a>
  <li><a href="investigations/merge-hi-tables/post-77-sera.html">Post-77 sera</a>
  <li><a href="investigations/merge-hi-tables/77-which-sera.html">Which post 77 sera to choose?</a>
  <li><a href="investigations/merge-hi-tables/predicting-new-pre-post-77-table.html">Predicting new-pre-post-77 titers from seq-t4 and seq-t5</a>
  <li><a href="investigations/merge-hi-tables/sequenced-strains-merge-take-6.html">Seq-t6</a>
  <li><a href="investigations/merge-hi-tables/misc-trans-77-experiments.html">Misc trans-77 tables in and out of seq-t4, t5 and t6</a>
  <li><a href="investigations/merge-hi-tables/attic-tables-onto-seq-t4.html">Add the attic tables (and 77 and 78) carefully onto seq-t4</a>
  <li><a href="investigations/merge-hi-tables/seq-t6-checking.html">Checking the seq-t6 1977-era</a>
  <p>
  <li><a href="investigations/merge-hi-tables/threshold-below-titer.html">Try making low titers into thresholds as a non-specific inhibition check</a>
  <li><a href="investigations/merge-hi-tables/sera-in-3d.html">Allow the sera to be in 3d, restrict the antigens to 2d (link is also in the row-col-adjust section above)</a>
  <p>
  <li><a href="investigations/shape/homologous-titer-normalization.html">Set col bases to homologous titers</a>
  <p>
  <li><h3>Explore the te77 kink for the nth time (2003-03)</h3>
  <li><a href="investigations/merge-hi-tables/seq-t8a-explore-kink-with-less-thans.html">Seq t8a, 2003-03, exploring the 77-kink with less thans</a>
  <li><a href="investigations/merge-hi-tables/seq-t8a-explore-kink-with-less-thans-part-two.html">Similar to above, but just the ag as link, not two sera also</a>
  <li><a href="investigations/merge-hi-tables/kink-history.html">Kink history</a>
  <li><a href="investigations/merge-hi-tables/pc-pre-post-77-onto-sequence.html">Orienting pre and post shape by the genetic map</a>
  <li><a href="investigations/shape-sequence/regression-without-trans-77-pairs.html">Regression without trans-77 pairs -- how much difference does it make?</a>
  <li><a href="investigations/merge-hi-tables/te77-link-other-labs.html">TE77 link, other labs</a>
  <li><a href="investigations/merge-hi-tables/seq-t8a-no-long-titers.html">Seq-t8a no long titers</a>
  <p>
  <li><h3>Getting the sequence antigenic map ready for publication (2003-03)</h3>
  <li><a href="investigations/merge-hi-tables/south-africa-table.html">South africa table and seq-t9 -- some stuff to figure out, leave for later</a>
  <li><a href="investigations/merge-hi-tables/seq-t9a-blobs.html">Seq-t9a -- blobs</a>
  <li><a href="investigations/merge-hi-tables/seq-t9a-multiple-optima-blobs.html">Seq-t9a -- identification of alternate optima by constant-force locii (blobs)</a>
  <li><a href="investigations/merge-hi-tables/seq-t9a-multiple-optima-by-single-point-randomization.html">Seq-t9a -- identification of alternate individual strain optima by randomizing and reoptimizing individal strains</a>
  <li><a href="investigations/merge-hi-tables/seq-t9a-multiple-optima-by-procrustes.html">Seq-t9a -- identification of alternate individual strain optima by comparing different stress solutions with procrustes</a>
  <li><a href="investigations/merge-hi-tables/seq-t9a-multiple-optima-summary.html">Seq-t9a -- identification of alternate optima summary</a>
  <li><a href="investigations/merge-hi-tables/seq-t9a-mod.html">Seq-t9a-mod -- gui mods and reoptimizes based on multiple-optima analyses</a>
  <li><a href="investigations/sequence-mds/299-mt35-mods.html">Sequence (299-mt35) mods -- gui mods and reoptimizes based on multiple-optima analyses (also linked below)</a>
  <li><a href="investigations/merge-hi-tables/seq-t9a-mod6-blobs.html">Seq-t9a-mod6 -- blobs</a>
  <li><a href="investigations/merge-hi-tables/seq-t9a-mod6-multiple-optima-by-single-point-randomization.html">Seq-t9a-mod6 -- identification of alternate individual strain optima by randomizing and reoptimizing individal strains (not finished, not needed)</a>
  <li><a href="investigations/merge-hi-tables/seq-t9a-mod10-sequence-correlation.html">Seq-t9a-mod10 -- sequence-shape-correlation</a>
  <li><a href="investigations/sequence-mds/2003-04-manuscript-sequences.html">2003-04 Dutch (and manuscrpt) genetic sequence set</a>
  <li><a href="investigations/sequence-mds/2003-04-maps.html">2003-04 Dutch (and manuscrpt) genetic sequence set maps</a>
  <li><a href="investigations/merge-hi-tables/seq-t9a-mod10-towards-publication-figures.html">Seq-t9a-mod10 -- towards publication figures</a>
  <li><a href="investigations/merge-hi-tables/seq-t9a-mod15-from-scratch.html">Seq-t9a-mod15 -- rerun from scratch</a>
  <li><a href="investigations/merge-hi-tables/fujian-in-merge.html">Fujian in merge (seq-t9a-mod17)</a>
  <li><a href="investigations/merge-hi-tables/fujian-in-merge-take-2.html">Fujian in merge -- take 2, after the 3rd Fujian table</a>
  <li><a href="investigations/merge-hi-tables/making-the-fujian-included-ms-figure.html">Finalizing the ms map with Fujian</a>
  <li><a href="investigations/merge-hi-tables/2004-02-16-publication-figures.html">Science submission figure making</a>
  <li><a href="investigations/merge-hi-tables/seq-t9a-mod27-show-ag-sr-connections.html">Science revision figure making (supplemental figure showing ag-sr connections)</a>
  <li><a href="investigations/merge-hi-tables/science-cover-candidates.html">Science cover candidates</a>
  <li><a href="investigations/merge-hi-tables/for-bjorn-seq-t9a-mod27.html">Remerge seq-t9a-mod27 in a way that we can merge additional data and get overall average titers (Derek and Bjorn 2009-11-03, updted 2009-11-25)</a>
  <li><a href="investigations/merge-hi-tables/seq-t9a-mod27-chain.html">Tables from the merge above for making chain for release (2010-04-05)</a>
  <li><a href="investigations/sequence-mds/2004-02-maps.html">Genetic map with Fujian (also linked in Sequence section below)</a>
  <li><a href="investigations/shape-sequence/2004-02-fujian.html">Genetic-antigenic correspondence for Fujian (2004-02) (also linked in sequence-shape section below)</a>
  <li><a href="investigations/shape-sequence/color-coded-alignment.html">Preliminary work on the color-coded alignment to support Ron</a>
  <li><a href="investigations/merge-hi-tables/are-tx77-and-ph82-one-cluster.html">Are TX77 and PH82 in the same cluster?</a>
  <li><a href="investigations/merge-hi-tables/seq-t9a-mod15-prototypes.html">Mark the prototypes in seq-t9a-mod15</a>
  <li><a href="investigations/merge-hi-tables/blobs-and-crosses.html">Try crosses inside the blobs to show the center</a>
  <li><a href="investigations/merge-hi-tables/seq-t9a-bootstrap.html">seq-t9a bootstraps (also listed in bootstrap section)</a>
  <li><a href="investigations/merge-hi-tables/seq-t9a-mod15-bootstraps.html">seq-t9a mod15 bootstraps (also listed in bootstrap section)</a>
  <li><a href="investigations/merge-hi-tables/seq-t9a-mod27-bootstraps.html">seq-t9a mod27 bootstraps (also listed in bootstrap section)</a>
  <li><a href="investigations/merge-hi-tables/seq-t9a-mod27-no-be89.html">seq-t9a-mod27 rerun without the be89 dead end</a>
  <li><a href="investigations/merge-hi-tables/seq-t9a-mod27-1d.html">seq-t9a-mod27 1D</a>
  <li><a href="investigations/seasons/rate-of-antigenic-evolution.html">Rate of antigenic evolution -- for publication figure</a>
  <li><a href="investigations/shape-sequence/cluster-gaps-single-linkage.html">Inter-cluster distances using single-linkage clusters</a> (<a href="investigations/shape-sequence/cluster-gaps.html">previously with Ron/Derek clusters</a>)
  <li><a href="investigations/shape-sequence/cluster-gaps-single-linkage-genetic.html">Inter-cluster distances using single-linkage clusters--genetic</a> 
  <li><a href="investigations/shape-sequence/cluster-gaps-k-means.html">Inter-cluster distances using k-means clusters--antigenic and genetic</a> 
  <li><a href="investigations/shape-sequence/cluster-gaps-k-means-mod15.html">Inter-cluster distances using k-means clusters--antigenic and genetic on mod15, and with new colors</a> 
  <li><a href="investigations/shape-sequence/cluster-gaps-k-means-mod27.html">Inter-cluster distances using k-means clusters--antigenic and genetic on mod27, and with new colors</a> 
  <li><a href="investigations/seasons/ag-evolution-all-strains.html">Inter-cluster distances using eyeballed clusters on all-strains</a> 
  <li><a href="investigations/shape-sequence/genetic-antigenic-distances.html">Genetic and antigenic distances from HK68 scatter plot</a> 
  <li><a href="investigations/shape/intercluster-distances.html">Check the cluster gaps in the raw HI</a>
  <li><a href="investigations/merge-hi-tables/alaska.html">Is the Finland clade the Alaska clade? -- Yes</a>
  <li><a href="investigations/shape-sequence/robins-codons.html">Shape correlation with Robin's codons</a>
  <li><a href="investigations/shape/direction-prediction.html">Direction prediction from prior immunity</a> (<a href="investigations/shape/direction-prediction-season-68-69.html">season-by-season here</a>)
  <li><a href="investigations/shape-sequence/mutations-at-145.html">Antgienic effects (on the map topolgy) of mutations at 145</a>
  <li><a href="investigations/shape-sequence/kn145-pairs.html">Estimating the magnitude of KN145 from pairs in the map with only KN145 between them</a>
  <li><a href="investigations/merge-hi-tables/wu-fi.html">Take a look at WU-FI, is FI a real clade?</a>
  <li><a href="investigations/merge-hi-tables/gu-only.html">Is the diversity in GU real, is there any hemisphering?</a>
  <li><a href="investigations/shape-sequence/fi-wu-again.html">Look with regression at 135KT as a possible wu-fi differentiating strain</a>
  <li><a href="investigations/shape-sequence/characteristic-antigenic-mutations.html">Characteristic antigenic mutations (supporting Ron) (linked below in sequence-shape also)</a>
  <li><a href="investigations/shape-sequence/characteristic-antigenic-mutations-mod15.html">Characteristic antigenic mutations - again, on mod15 (linked below in sequence-shape also)</a>
  <li><a href="investigations/shape-sequence/characteristic-antigenic-mutations-mod27.html">Characteristic antigenic mutations - again, on mod27 (linked below in sequence-shape also)</a>
  <li><a href="investigations/shape-sequence/antigenic-codon-parameter-estimates.html">The antigenic effect of the cluster-defining substitutions</a>
  <li><a href="investigations/merge-hi-tables/g-b-interdigitation.html">G-B interdigitation (for later)</a>
  <p>
  <li><a href="investigations/merge-hi-tables/shape-as-tree.html">Shape as tree</a>
  <p>
  <li><h3>Routine surveillance</h3>
  <li><a href="investigations/merge-hi-tables/tab86.html">Add Tab86 (the Dutch 2003/2004 season epi table) to seq-t9a-mod27 (the Science ms ag map)</a>
  <li><a href="investigations/merge-hi-tables/tab87.html">Add Tab87 (an earlier table set to Hay, for us to look at NL/17/03) to above table</a>
  <li><a href="investigations/merge-hi-tables/tab88.html">Add Tab88 to above table</a>
  <li><a href="investigations/merge-hi-tables/jan-april-2005/main.html">EMC Tables from Feb-Apr 2005</a>
  <li><a href="investigations/merge-hi-tables/jan-april-2005/post-ron.html">Some merges of the above tables after Ron cleaned them up</a>
  </UL>     


<H1> Sequence</H1>
  <UL>
  <li><a href="data/all-seq/2001-06/workings.html">The differences between the 242 and 315 sequences</a>
  <p>
  <li><a href="investigations/sequence-mds/nl-pro-runs.html">NL protein sequence MDS</a>
  <li><a href="investigations/sequence-mds/sequence-no-long-distances.html">Sequence MDS -- take2, no long distances</a>
  <li><a href="investigations/sequence-mds/groups-vs-all-take-2.html">MDS of groups compared to MDS of all together (take 2)</a>
  <p>
  <li><a href="investigations/sequence-mds/hamming.html">Hamming (including pivot investigation with procrustes clusterer)</a>
  <li><a href="investigations/sequence-mds/kimura.html">Kimura</a>
  <li><a href="investigations/sequence-mds/PAM.html">PAM</a>
  <li><a href="investigations/sequence-mds/barker.html">Category (Barker)</a>
  <li><a href="investigations/sequence-mds/eisenberg.html">Eisenberg</a>
  <li><a href="investigations/sequence-mds/asl/Escore.html">ASL on Eisenberg scores</a>
  <li><a href="investigations/sequence-mds/sqrt-hamming.html">Sqrare root Hamming</a>
  <p>
  <li><a href="investigations/hammerhead-conjecture/initial-thoughts.html">The "Hammerhead Conjecture"</a>
  <li><a href="investigations/hammerhead-conjecture/first-runs.html">Initial investigation</a>
  <p>
  <li><a href="investigations/sequence-mds/2002-02.html">2002-02, 299 strains (includind the new early strains)</a>
  <li><a href="investigations/sequence-mds/2002-02-take-2.html">2002-02, 299 strains -- take 2 June 2002 (use dim anneal)</a>
  <p>
  <li><a href="investigations/sequence-mds/299-strains-plus.html">making 2002-08 (2002-02 eur strains plus non-eur from 2001-06)</a>
  <li><a href="investigations/sequence-mds/2002-08-threshold-runs-new.html">2002-08 threshold runs</a> (<a href="investigations/sequence-mds/2002-08-threshold-runs.html">with two strains different, older data</a>)
  <li><a href="investigations/sequence-mds/thresholds.html">Thresholds</a>
  <li><a href="investigations/sequence-mds/end-expansion.html">End expansion</a>
  <li><a href="investigations/sequence-mds/multiple-thresholds.html">Multiple thresholds</a>
  <p>
  <li><a href="investigations/sequence-mds/299-mt35-mods.html">Sequence (299-mt35) mods -- gui mods and reoptimizes based on multiple-optima analyses (also linked above)</a>
  <p>
  <li><a href="investigations/sequence-mds/2002-10.html">2002-10 sequence set, sanity check</a>
  <li><a href="investigations/sequence-mds/2002-10-eur-only.html">2002-10 sequence set, which are the strains we have HI for, and thus which strains make "tree-2"</a>
  <li><a href="investigations/sequence-mds/373-mt30.html">2002-10 sequence set, multiple threshold 30 runs and bootstraps</a>
  <p>
  <li><a href="investigations/sequence-mds/2003-04-manuscript-sequences.html">2003-04 Dutch (and manuscrpt) sequence set</a>
  <li><a href="investigations/sequence-mds/2003-04-maps.html">2003-04 Dutch (and manuscrpt) genetic sequence set maps</a>
  <p>
  <li><a href="investigations/sequence-mds/2004-02-manuscript-sequences.html">2004-02 Dutch sequence set</a>
  <li><a href="investigations/sequence-mds/2004-02-maps.html">Genetic map with Fujian (also linked in Shape section above for ms completion URLS)</a>
  <p>
  <li><a href="investigations/sequence-mds/dnaml.html">Compare DNA-ML distance with Hamming and post-MDS distances</a>
  <li><a href="investigations/sequence-mds/dnaml2.html">Make "phylogenetic" maps</a>
  <li><a href="investigations/sequence-mds/dnaml-for-science.html">Reordering the dnaml trees for the Science submission</a>
  </UL>     

<H1> Comparison of sequence and shape</H1>
  <UL>
  <li>Some correlations on distance matrices towards the bottom of <a href="investigations/piecing-hi-tables/start-looking-at-jan-tables.html">Feeling out the individual Jan tables</a>
  <li>Some scatter plots on raw (un-MDS'ed) data in <a href="investigations/piecing-hi-tables/by-big-merge/by-big-merge.html">Workings to set up the big merge</a>
  <p>
  <li><a href="investigations/groups/play-with-groups.html">Play with groups</a>
  <p>
  <li><a href="investigations/row-col-adjusts/comparison-of-sequence-and-shape-take-2.html">Comparison of sequence and shape</a>  
  <li><a href="investigations/shape-sequence/different-sequence-measures.html">Different sequence measures</a>
  <p>
  <li><a href="investigations/shape-sequence/no-dead-end.html">No dead end</a>
  <p>
  <li><a href="investigations/shape-sequence/oct-2001.html">67 tables (oct 2001)</a>
  <li><a href="investigations/shape-sequence/nov-2001.html">67 tables (nov 2001)</a>
  <li><a href="investigations/shape-sequence/procrustes-sequenced-shape.html">sequence - shape, sequence run with dim anneal, and shape from take-4 (compare with procrustes) (june 2002)</a>
  <p>
  <li><a href="investigations/shape-sequence/selecting-new-strains-to-sequence.html">Selecting new strains to sequence (oct 2001)</a>
  <li><a href="investigations/shape-sequence/vaccine-tables.html">Vaccine sequence tables (supporting Ron's work on vaccine HI tables)</a>
  <p>
  <li><a href="investigations/shape-sequence/genetic-antigenic-comparison-initial.html">Genetic antigenic comparison, generation of data (December 2002), and ag effect of point mutations</a>
  <li><a href="investigations/shape-sequence/genetically-identical-strains.html">Genetically identical strains</a>
  <li><a href="investigations/shape-sequence/aa-changes-per-position.html">AA changes per position</a>
  <li><a href="investigations/shape-sequence/antigenic-genetic-regression.html">First cut at using regression to calculate the effects of each mutation</a>
  <li><a href="investigations/shape-sequence/antigenic-genetic-regression-part-2.html">Look at using upto 1 2 3 4 6 and 10 mutations</a>
  <li><a href="investigations/shape-sequence/antigenic-genetic-regression-comparison-of-t7-dead-end-and-pret8.html">1 mutation difference between t7-dead-end and pre-t8</a>
  <li><a href="investigations/shape-sequence/model-fitness.html">First look at model fitness (including compare HD and regressed distance)</a>
  <li><a href="investigations/shape-sequence/no-single-mutations.html">Run without single mutations, and see how well they are predicted</a>
  <li><a href="investigations/shape-sequence/regression-using-r.html">Regresson using R</a>
  <li><a href="investigations/shape-sequence/non-zero-ag-dist-when-zero-mutations.html">Forcing a 0 intercept</a>
  <li><a href="investigations/shape-sequence/non-zero-theory.html">Examine the thoery about short genetic distances having biased average prediction errors</a>
  <li><a href="investigations/shape-sequence/non-zero-theory-part-2.html">Examinging the 0.25 asymptote in the "no negative separation effect" above</a>
  <li><a href="investigations/shape-sequence/non-zero-theory-part-3.html">Examinging the 0.25 asymptote in the "no negative separation effect" above (do again, this time finding "2nd dimension bias" (Dec 2004))</a>
  <li><a href="investigations/shape-sequence/non-zero-theory-part-4.html">Examinging the 0.25 asymptote in the "no negative separation effect" above (do again, this time finding "2nd dimension bias" with circular not rectangular error (Dec 2005))</a>
  <li><a href="investigations/shape-sequence/non-zero-theory-part-5.html">Examinging the 0.25 asymptote in the "no negative separation effect" above (do again, this time finding "2nd dimension bias" (circular) and look at expected true distance given an observed distance(Dec 2005))</a>
  <li><a href="investigations/shape-sequence/unbias-the-antigenic-distances.html">Unbias the antigenic distances according to the "no negative separation effect" described above</a>
  <p>
  <li><a href="investigations/shape-sequence/independence-among-samples.html">Comparing hamming and post-regression genetic distances and initial testing of independence among samples</a>
  <li><a href="investigations/shape-sequence/first-look-for-interactions.html">First look for interaction -- ONGOING</a>
  <li><a href="investigations/shape-sequence/regression-without-trans-77-pairs.html">Regression without trans-77 pairs -- how much difference does it make?</a>
  <li><a href="investigations/shape-sequence/check-kn145-as138.html">Check the data used in the KN145 AS138 interaction</a>
  <li><a href="investigations/shape-sequence/collinear-testing.html">Feel out what is happening with collinearity and aliasing</a>
  <li><a href="investigations/shape-sequence/alan-sync.html">Sync with Alan</a>
  <p>
  <li><a href="investigations/shape-sequence/dan-intro.html">Dan intro</a>
  <p>
  <li><a href="investigations/shape-sequence/seq-t8-299.html">Seq t8 and 299-mt35 (jan 2003)</a>
  <li><a href="investigations/shape-sequence/early-wu.html">Early WU strains (the effect of KN145)</a>
  <li><a href="investigations/shape-sequence/characteristic-antigenic-mutations.html">Characteristic antigenic mutations (supporting Ron) (also liked above with the ms links)</a>
  <li><a href="investigations/shape-sequence/characteristic-antigenic-mutations-mod15.html">Characteristic antigenic mutations -- on mod 15 (also liked above with the ms links)</a>
  <li><a href="investigations/shape-sequence/characteristic-antigenic-mutations-mod27.html">Characteristic antigenic mutations -- on mod 27 (also liked above with the ms links)</a>
  <p>
  <li><a href="investigations/merge-hi-tables/alan-split-clade.html">Some figures for Alan's split clade</a>
  <p>
  <li><a href="investigations/shape-sequence/step/step.html">Using step</a>
  <p>
  <li><a href="investigations/shape-sequence/2004-02-fujian.html">Genetic-antigenic correspondence for Fujian (2004-02) (also linked from Shape section above, for ms completion urls)</a>
  <p>
  <li><a href="investigations/shape-sequence/plot-substitutions.html">Substitutions causing the turn?</a>
  <p>
  <li><h3>Ag effect of substitutions, from November 2004</h3>
  <li><a href="investigations/shape-sequence/preparing-for-ag-effect-experiments.html">Preparing for ag-effect experiments</a>
  <li><a href="investigations/shape-sequence/seq-t9a-mod27-sub-and-pos-upto-25-mutations/index.html">Same as above, but out to 25 substitutions</a>
  <li><a href="investigations/shape-sequence/seq-t9a-mod27-sub-and-pos-upto-10-mutations/index.html">Same as above, but out to 10 substitutions</a>
  <li><a href="investigations/shape-sequence/dan12.html">Dan: spot and span, out to 10 mutations</a>
  <p>
  <h3>Ana from April 2006</h3>
  <li><a href="investigations/shape-sequence/ana/to-ana-derek-runs/index.html">Getting baseline agreement of results</a>
  <p>
  <h3>Again, with maximum likelihood unbiasing (April 2006)</h3>
  <li><a href="investigations/shape-sequence/unbias-the-antigenic-distance-using-mle.html">Initial runs (2007-01-10)</a>
  <li><a href="investigations/shape-sequence/new-regression-diagnostics.html">35 (full dataset) baseline (no distance unbias from link above again with latest diagnostics) (2009-02-06)</a>
  <li><a href="investigations/shape-sequence/unbias-the-antigenic-distance-using-mle-initial-analysis.html">Analysis of unbiasing methods (none, sequence-based, ML) (2007-04) </a>
  <li><a href="investigations/shape-sequence/unbias-the-antigenic-distance-using-mle-not-unbiased-predictions.html">Don't unbais predictions (2007-05-09) </a>
  <li><a href="investigations/shape-sequence/unbias-the-antigenic-distance-using-mle-test-with-no-biased.html">Compare unbaising with removing short distance (biased) strains. Remove ad<2 and hd<5 (2007-04) </a>
  <li><a href="investigations/shape-sequence/unbias-the-antigenic-distance-using-mle-test-without-singleton-pairs.html">Compare with and without pairs involved in singletons (2007-04) </a>
  <li><a href="investigations/shape-sequence/unbias-the-antigenic-distance-using-mle-15-20-25.html">Pairs out to 5 10 15 20 and 25 HD (2007-04) </a>
  <li><a href="investigations/shape-sequence/unbias-the-antigenic-distance-using-mle-50-50-train-test.html">Split dataset 50-50 into train-test instead of our usual 90-10 (2007-04) </a>
  <li><a href="investigations/shape-sequence/unbias-the-antigenic-distance-using-mle-predict-strains.html">Predict whole strains (2007-04) </a>
  <p>
  <li><a href="investigations/shape-sequence/unbias-the-antigenic-distance-using-mle-se-vs-n.html">Plot SE vs sqrt N (2007-07) </a>
  <li><a href="investigations/shape-sequence/unbias-the-antigenic-distance-using-mle-se-vs-n-split-by-cluster.html">Split QR179 and DG124 by cluster (2007-07) </a>

  <p>
  <li><a href="investigations/shape-sequence/test-aas.html">Substitutions to test regression (2007-07) </a>
  <li><a href="investigations/shape-sequence/test-aas-check-transitions.html">Check substitutions in different custers to test which backbone we should use (2007-07-25) </a>
  <p>
  <h3>Try nonlinear regression instead of unbiasing</h3>
  <li><a href="investigations/shape-sequence/nls-mock-data.html">Initial run, mock data (2007-08-04) </a>
  <li><a href="investigations/shape-sequence/nls-hand-managed-real-data.html">Initial run, real data (2007-08-04) </a>
  <li><a href="investigations/shape-sequence/nls-35yr-culled.html">Non-linear regression (nls) on the 35yr culled dataset (2008-11-24) (also linded below in section on testing hypotheses of cause of negative parameter estimates)</a>
  <li><a href="investigations/shape-sequence/alternate-nls-models.html">NLS models that incorportate an intercept (2009-01-10) (also linked below)</a>
  <p>
  <h3>Spot and span</h3>
  <li><a href="investigations/shape-sequence/spot-and-span.html">Spot and span (2007-10-09) </a>
  <li><a href="investigations/shape-sequence/spot-and-span-cluster-and-strain-sensitivity.html">Spot and span separating most common strain mutation and subst (2007-11-01) </a>
  <li><a href="investigations/shape-sequence/spot-span-visualization.html">Spot and span raw data upto 25 mutations (2007-11-09) </a>
  <li><a href="investigations/shape-sequence/span-on-culled.html">Span on the culled 35 year dataset (also linked below with the culled dataset links) (2008-12-22) </a>
  <li><a href="investigations/shape-sequence/span-backwards-on-culled.html">Backwards span on the culled 35 year dataset (also linked below with the culled dataset links) (2008-12-22) </a>
  <p>
  <li><a href="investigations/h5/regression/index.html">H5 single mutants (2007-12-22) </a>
  <p>
  <li><a href="investigations/shape-sequence/no-distance-unbias/index.html">Best current 35-year regression (2008-01-26) </a>
  <p>
  <h3>Regression on SSM data</h3>
  <li><a href="investigations/shape-sequence/ssm-regression/index.html">Inital runs (2008-07-10) </a>
  <li><a href="investigations/strain-selection-meeting/database/combined/investigations/2008-sh-misc/regressions/index.html">Ag effect of aa substs on SSM data (2007.5 to 2008.5 subsets, separate for each lab) (2008-08-24)</a>
  <p>
  <h3>Regression on rabies data</h3>
  <li><a href="investigations/rabies/ag-effect-of-aa-substs/index.html">Inital runs (2008-08-19) </a>
  <p>
  <h3>Testing hypothesis of cause of negative parameter estimates</h3>
  <li><a href="investigations/shape-sequence/2d-3d-comparison.html">2D 3D 35-year map regression comparison (2008-11-18)</a>
  <li><a href="investigations/shape-sequence/robust-regression.html">Robust regression (2008-10-23) </a>
  <li><a href="investigations/shape-sequence/compare-out-to-10-and-25-mutations.html">Compare out to 10, and out to 25 mutations (2008-11-29)</a>
  <p>
  <li><a href="investigations/shape-sequence/proportion-strains-involved.html">Do regression on subsets of the data for which there is less dependence on single strains (2008-11-18)</a>
  <li><a href="investigations/shape-sequence/cull-further.html">Cull the 35 year data to get a set of substs that are potentially more reliable (2008-11-22)</a>
  <li><a href="investigations/shape-sequence/cull-further-to-25-mutations.html">Cull the 35 year data to get a set of substs that are potentially more reliable -- now out to 25 mutations (2008-11-30)</a>
  <li><a href="investigations/shape-sequence/span-on-culled.html">Span on the culled 35 year dataset (also linked above in the spot-span section) (2008-12-22) </a>
  <li><a href="investigations/shape-sequence/span-backwards-on-culled.html">Backwards span on the culled 35 year dataset (also linked above in the spot-span section) (2008-12-22) </a>
  <p>
  <li><a href="investigations/shape-sequence/step/step-again.html">Using step again, with Rob (2008-11-10)</a>
  <li><a href="investigations/shape-sequence/step/step-again-HK68.html">stepAIC on HK68, with Rob (2008-11-15)</a>
  <li><a href="investigations/shape-sequence/step/step-again-35YR.html">stepAIC on 35 year data, with Rob (2008-11-18)</a>
  <li><a href="investigations/shape-sequence/step/35yr-step-backwards-on-substs.html">35yr step backwards on substs (2008-11-18)</a>
  <li><a href="investigations/shape-sequence/step/step-again-35yr-culled.html">35yr step experiments on culled (1.0 20) dataset (2008-11-22)</a>
  <li><a href="investigations/shape-sequence/step/subst-subst-interaction-term-example.html">35yr step experiments on culled (1.0 20) dataset -- tracking down the details of the first interactions term that stepAIC finds (2008-12-14)</a>
  <li><a href="investigations/shape-sequence/leave-one-subst-out.html">35yr step experiments on culled (1.0 20) dataset -- tracking down the details of the first interactions term that stepAIC finds.  Look for "pliability" by looking at effect of leaving out substs and strains (2009-03-10)</a>
  <li><a href="investigations/shape-sequence/step/order-of-adding-interaction-terms.html">Is there an effect size pattern associated with the order stepAIC adds terms (2008-11-26)</a>
  <li><a href="investigations/shape-sequence/step/excluding-strain-number-interaction-terms.html">Excluding interaction terms with strain number greater than X (2008-12-21)</a>
  <p>
  <li><a href="investigations/shape-sequence/remove-short-distances-to-test-non-zero-distance-bias.html">Remove short distances to test non zero distance bias (2008-11-23)</a>
  <li><a href="investigations/shape-sequence/test-intercept-as-cause-for-negative-distances.html">Test intercept as cause for negative distances (2008-11-23)</a>
  <li><a href="investigations/shape-sequence/nls-35yr-culled.html">Non-linear regression (nls) on the 35yr culled dataset (2008-11-24)</a>
  <li><a href="investigations/shape-sequence/nls-on-short-distance-trimmed-datasets.html">NLS on the short-distance (both antigenic and genetic) trimmed datasets (2009-01-02)</a>
  <li><a href="investigations/shape-sequence/alternate-nls-models.html">NLS models that incorportate an intercept (2009-01-10) (also linked in nls section above)</a>
  <p>
  <li><a href="investigations/shape-sequence/train-test-set-proportion-test.html">Train/test set proportion test (2008-11-23)</a>
  <p>
  <li><a href="investigations/shape-sequence/step/cluster-interaction-terms.html">Add cluster-pair as an interaction term (2008-11-25)</a>
  <li><a href="investigations/shape-sequence/step/cluster-interaction-terms-to-25.html">Add cluster-pair as an interaction term, out to 25 mutations (2008-11-30)</a>
  <p>
  <li><a href="investigations/shape-sequence/comparison-with-empirical-estimates.html">Comparison with empirical estimates (2008-12-02)</a>
  <p>
  <li><a href="investigations/shape-sequence/interdependence.html">Initial incomplete visualization of interdependece among substitutions (2008-12-02)</a>
  <p>
  <li><a href="investigations/shape-sequence/bjorn-measurements.html">Bjorn measurements of regression parameters</A> (from <A href="https://notebooks.antigenic-cartography.org/bjorn/empirical-regression/empirical_distaces/summary_all_distances.txt">here</A>) (2008-12-24)</a>
  <p>
  <h3>H5 test case</h3>
  <li><a href="investigations/shape-sequence/h5/importing-h5-map.html">Importing the H5 map (2008-12-06)</a>
  <li><a href="investigations/shape-sequence/h5/h5-regression.html">H5 test case regression (2008-12-06)</a>
  <li><a href="investigations/shape-sequence/h5/include-zero-genetic-distances.html">Compare with and without zero genetic distances (2008-12-06)</a>
  <li><a href="investigations/shape-sequence/h5/subst-subst-interaction-terms.html">Subst-subst interaction terms (2008-12-08)</a>
  <li><a href="investigations/shape-sequence/h5/h5-map-variants.html">H5 map variants (and original), regression, and step (2008-12-16)</a>
  <li><a href="investigations/shape-sequence/h5/testing-positive-interaction-hypothesis.html">Test Great Wall hypothesis for positive interaction terms (2008-12-16)</a>
  <li><a href="investigations/shape-sequence/h5/intercept-se-relationship.html">Explore the intercept-SE relationship (leave out some observations from H5 data to get other SEs) (2008-12-24)</a>
  <li><a href="investigations/shape-sequence/h5/h5-neutrals-genetically-identical.html">H5 neutrals genetically identical (2008-01-07)</a>
  <h3>Post negative-parameter-estimate thinking</h3>
  <li><a href="investigations/shape-sequence/proportion-of-intercept.html">Proportion of intercept, intercept for different HD, general thinking about intercept (all in workings) (2009-03-14)</a>
  <li><a href="investigations/shape-sequence/iterate-regression.html">Iterate regression in attempt to remove non-neg-distance bias (2009-03-14)</a>
  <li><a href="investigations/shape-sequence/blob-error.html">Weighted regression using blob error (2009-03-23)</a>
  <h3>TODOs</h3>
  <li><a href="investigations/shape-sequence/todo.html">TODOs (2008-11-21)</a>

  <h3>With Hannah (Summer 2011)</h3>
  <li><a href="investigations/shape-sequence/hannah-demo.html">Hannah demo (2011-07-12)</a>

  <br><br><br>WHEN ITEMS ARE ADDED TO THE ABOVE SECTION, ALSO ADD TO THE INDEX PAGE IN THE shape-sequence DIRECTORY <A href="investigations/shape-sequence/index.html">here</A><br><br>

  <h3>NA regression (with Ron Kim and Terry (2015-05-29)</h3>
  <li><a href="investigations/NA-regression/run-1/index.html">First NA regression, on full length NA, no unbiasing, upto 10 substs (with Kim and Ron) (2015-05-19)</a>
  <li><a href="investigations/NA-regression/NA-regression-full-no-10-substs-limit.html">Running regression on the NA map, with full-length sequences and norestriciton on num substs between strains form making pairs (2015-06-06)</a>  <li><a href="investigations/NA-regression/NA-stripped.html">NA regression on stripped NA to just the fixed locii, and comparison of upto 10, 7, 5, and 3 substs (2015-05-29)</a>
  <li><a href="investigations/NA-regression/NA-regression-on-neighboring-pairs-of-clusters.html">NA regression on stripped NA: just between neighboring pairs of antigenic clusters defined by Terry (k-means k=6)(2015-06-09)</a>
  <p>
  <li><a href="investigations/NA-regression/HI-stripped.html">HA regression on the 2004 map stripped to just the fixed locii (2015-05-29)</a>

    


  </UL>     

<H1> HI prediction from sequence</H1>
  <UL>
  <li><a href="investigations/hi-prediction/hi-prediction-from-sequence.html">HI prediction from sequence</a>
  <li><a href="investigations/hi-prediction/missmatches.html">Missmatches</a>
  <p>
  <li><a href="investigations/shape-sequence/procrustes.html">Procrustes</a>
  <p>
  <li><a href="investigations/hi-prediction/initial-sequence-design.html">Initial look at sequence design</a>
  <p>
  <li><a href="investigations/hi-prediction/betweenness-notes.html">Worries about predicting new clades and true antigenic maps</a>
  <p>
  <h3>August 2002</h3>
  <li><a href="investigations/hi-prediction/setting-up-for-aug-2002-predictions.html">Setting up data for aug 2002 predictions of jo-wu HI</a>
  <p>
  <li><a href="investigations/shape-sequence/predicting-antigenic-distance-from-hamming-distance.html">Predicting antigenic distance from Hamming distance (just from sequence-shape regression, deliberately simple for first ms)</a>
  </UL>


<H1> HI prediction from HI</H1>
  <UL>
  <li><a href="investigations/hi-prediction-from-hi/hi-prediction-from-hi.html">HI prediction from HI</a>
  <li><a href="investigations/hi-prediction-from-hi/initial-predictions.html">Initial predicitons</a>
  <li><a href="investigations/hi-prediction-from-hi/more-initial-predictions.html">More initial predictions</a>
  <li><a href="investigations/hi-prediction-from-hi/num-dimensions.html">Number of dimensions</a>
  <li><a href="investigations/hi-prediction-from-hi/leave-one-out.html">Leave-one-out on tab2</a>
  <li><a href="investigations/hi-prediction-from-hi/prediction-fitting-systematic-errors.html">Prediction fitting systematic errors (concentric circles)</a>
  <p>
  <h3>August 2002</h3>
  <li><a href="investigations/hi-prediction-from-hi/dimensional-analysis-seq-t4.html">Dimensional analysis seq-t4</a>
  <li><a href="investigations/hi-prediction-from-hi/setting-up-for-jo-wu-predictions.html">Setting up for jo-wu predictions</a>
  <li><a href="investigations/hi-prediction-from-hi/blind-prediction.html">Setting up, submitting, and recovering the jo-wu predictions</a>
  <li>(<a href="investigations/hi-prediction-from-hi/jo-wu-prediction.html">Misc workings on JO-WU</a>)
  <li><a href="investigations/hi-prediction-from-hi/jo-wu-predictions.html">Prediction of JO-WU table results</a>
  <li><a href="investigations/hi-prediction-from-hi/jo-wu-higher-than-homologous-predictions.html">Prediction of JO-WU higher than homologous titers</a>
  <li><a href="investigations/hi-prediction-from-hi/jo-wu-seeing-the-difference-prediction.html">Prediction of JO-WU: predictions when the serum cannot see the true antigenic difference</a>
  <li><a href="investigations/hi-prediction-from-hi/jo-wu-seeing-the-difference-prediction-single-predictions.html">Prediction of JO-WU: predictions when the serum cannot see the true antigenic difference (with single prediction, not double prediction)</a>
  <li><a href="investigations/hi-prediction-from-hi/jo-wu-symmetry-predictions.html">Prediction of JO-WU: symmetry predictions</a>
  <p>
  <li><a href="investigations/hi-prediction-from-hi/seq-t6-jw-jo-wu.html">The JO-WU region in seq-t6-jw   (<A href="investigations/hi-prediction-from-hi/jw-season-89-90.html">season-by-season here</A>)</a>
  <li><a href="investigations/hi-prediction-from-hi/seq-t6-jw-93-94-season.html">Look at the 93-94 season of the JO-WU table</a>
  <p>
  <li><a href="investigations/hi-prediction-from-hi/cross-validation.html">Cross-validation</a>
  <p>
  <li><a href="investigations/rabies/prediction/prediction.html">Rabies predictions with Dan (to determine accuracy and dimension) (2007-08-30)</a>  
  <li><a href="investigations/equine-aht/prediction/prediction.html">Equine flu predictions with Nic (to determine accuracy and dimension) (2007-09-01)</a>  
  <li><a href="investigations/rabies/prediction/auto-prediction-test-2a/index.html">Rabies predictions with Dan (to determine accuracy and dimension) (automated) (2008-12-22)</a>  
  </UL>


<H1> Structure</H1>
  <UL>
  <li><a href="investigations/h3-structure/data-to-david-burke.html">Data to David Burke (July 2006)</a>
  <p>
  <li><a href="investigations/h3-structure/demo/index.html">Demo</a>
  <li><a href="investigations/h3-structure/map-from-regredssion.html">Initial work on antigenic map from regression (with David 2007-10-31)</a>
  </UL>


<H1> Fitness</H1>
  <UL>
  <li><a href="investigations/fitness-exchange/plaque-size-distributions.html">Plaque size distributions (2008-06-01)</a>
  <p>
  <li><a href="investigations/fitness-exchange/sir.html">Competition at moment fitness exchange--simple epi model (with Colin) (2008-06-12)</a>
  <p>
  <li><a href="investigations/fitness-exchange/skinny-bone-tree.html">Skinny-bone tree ag map (2008-09-01)</a>
  <p>
  <li><a href="investigations/fitness-exchange/miranda-images/miranda-images.html">Miranda images from plaque assay experiment (2010-11-17)  (carful, 70 meg of images)</a>
  </UL>


<H1> Plaque assay unbiasing</H1>
  <UL>
  <li><a href="investigations/plaque-assay-bias/sparse-seed-number.html">Sparse seed number (2010-10-22)</a>  
  <li><a href="investigations/plaque-assay-bias/bias-in-practice.html">Bias in practice when being sparse (2010-10-24)</a>  
  <li><a href="investigations/plaque-assay-bias/how-much-variation-in-different-plates.html">How much variation from plate to plate in estimate of growth rate (2010-10-29)</a>  

  <p>
  <li><a href="investigations/plaque-assay/index.html">Index page for working with Ramona (2011-03-21)</a>
  <p>
  <li><a href="investigations/plaque-assay/annotate-plates.html">Annotate plates to explore typhoon plaque size estimation (2011-03-15 with Ramona)</a>
  <li><a href="investigations/plaque-assay/relative-plaque-size-plots.html">Relative plaque size plots (2011-03-15 with Ramona)</a>
  <li><a href="investigations/plaque-assay/initial-regressions.html">Initial plaque assay regressions (2011-03-20 with Ramona)</a>
  <li><a href="investigations/plaque-assay/unbias.html">Unbias by excluding plaques within 2x of max radius of a plaque on a plate(2011-03-22)</a>
  <li><a href="investigations/plaque-assay/rates.html">Rates, an addendum to Ramona's page (<A href="https://notebooks.antigenic-cartography.org/ramona/plaque-assay/estimate-rates.html">here</A>) (with Sarah and Colin 2011-03-30-approx)</a>
  </UL>


<H1> Prediction of heavy hitters</H1>
  <UL>
  <li><a href="investigations/hh-prediction/h3-7-locations.html">H3, 7 locations, cf Bjorn (2010-11-24)</a>  
  <li><a href="investigations/hh-prediction/actual-cluster-reps.html">How many neighbors of the actual cluster reps? (2010-11-24)</a>  
  <li><a href="investigations/hh-prediction/aa-nucs.html">The nucleotides in the cluster representatives (2013-09-20)</a>  
  <li><a href="investigations/hh-prediction/miyata-distance.html">Sort the aa neighbors by some measure of chemical distance, in this case Miyata distance (2010-11-25)</a>  
  <li><a href="investigations/hh-prediction/miyata-distance-2.html">Further Miyata distance analyses (with David, Chris, Kyle) (2010-12-08)</a>  
  <li><a href="investigations/hh-prediction/aa-usage.html">Amino acid usage (2010-12-05)</a>  
  <li><a href="investigations/hh-prediction/how-special-are-the-bjorn-7.html">How unusual is the Bjorn 7 use of AAs? (2010-12-06)</a>  
  <li><a href="investigations/hh-prediction/miyata-num-nuc-relationship.html">Miyata distance num-nucs correlation (2014-01-09)</a>
  <li><a href="investigations/hh-prediction/miyata-num-nucs.html">Miyata maps showing num nucs between aas (2014-01-15)</a>
  <li><a href="investigations/hh-prediction/miyata-color-code-map.html">Color code, and orient, Miyata distance map by biochemical properties of the aa (2014-02-01)</a>
  <p>
  <li><a href="investigations/hh-prediction/FitnessExchange/fitness.html">B7 prediction experiment, aa's per location (David's work, posted here as he is offline) (2013-09-08)</a>  
  <li><a href="investigations/hh-prediction/hh-prediction-notes.html">Experimental results 1, musings (2013-09-11)</a>  
  <li><a href="investigations/hh-prediction/plaque-sizes.html">Rough ag dist plaque size correlation (2013-11-14)</a>
  <p>
  <li><a href="investigations/hh-prediction/hh-wobble.html">Rough workings: Initial pass at coloring trees to look for wobble in the B7 (2013-11-23)</a>  
  <li><a href="investigations/hh-prediction/running-ca04-wi05.html">Rough workings: Running the CA04-WI05 tree (2013-11-25)</a>  
  <li><a href="investigations/hh-prediction/running-wi05-pe09.html">Rough workings: Running the WI05-PE09 tree (2013-11-25)</a>  
  <li><a href="investigations/hh-prediction/hh-wobble-final-two-clusters.html">Rough workings: Coloring trees for the final two clusters to add onto the data in link above (2013-11-26)</a>  
  <li><a href="investigations/hh-prediction/hh-wobble-recolor-trees.html">Rough workings: 2nd pass at coloring trees to look for wobble in the B7(2013-11-29)</a>  
  <li><a href="investigations/hh-prediction/b7-aa-usage.html">Rough B7 AA usage for determining tree coloring (2013-11-30)</a>  
  <li><a href="investigations/hh-prediction/hh-wobble-recolor-trees-2.html">Recoloring the trees to look for wobble in the B7, 3rd pass (2013-12-10)</a>  
  <li><a href="investigations/hh-prediction/hh-wobble-recolor-trees-3.html">Recoloring the trees to look for wobble in the B7, 4th pass, including stitching the individual trees into one file (2013-12-23)</a>  
  </UL>


<H1> Seasons</H1>
  <UL>
  <p>
  <li><a href="investigations/seasons/year-by-year.html">Shape year by year (on sequenced strain merge take-4) (part 1</a>, <a href="investigations/seasons/year-by-year-part-2.html">part 2</a>, <a href="investigations/seasons/year-by-year-part-3.html">part 3</a>, <a href="investigations/seasons/year-by-year-part-4.html">part 4</a>)
  <li><a href="investigations/seasons/all-strains.html">All strains calculations</a>  (<a href="investigations/seasons/all-strains-2002-02-26-upto-season-68-69-rerun.html"> results </a>).
  <li><a href="investigations/seasons/changes.html">How a season's strains move as more seasons data are added</a> (<a href="investigations/seasons/season-change-maps.html">season change maps</a>)
  <li><a href="investigations/seasons/strains-per-season.html">Number of strains per season</a>
  <li><a href="investigations/seasons/bootstraps.html">Bootstraps with the same number of strains per season (also in the bootstraps section below)</a>
  <p>
  <li><a href="investigations/merge-hi-tables/2002-2003-season.html">2002-2003 season</a>
  <p>
  <li><a href="investigations/seasons/vaccine-to-epi-distance.html">Vaccine to epi distances (initially for Bettie)</a>  
  </UL>     


<H1> Bootstraps</H1>
  <UL>
  <p>
  <li><a href="investigations/seasons/bootstrap-procrustes-kmeans-2-and-3-clade-comparison.html">Comparison of 2- and 3-clade clustering of the sequences strain bootstraps</a>
  <li><a href="investigations/seasons/bootstrap-n-strains-cluster-rerun-comparison.html">Comparison of 2 sets each of 3 runs of the clusterer (for the n-sample 2 clusters of seq-t4 and seq-t5)</a>
  <li><a href="investigations/seasons/bootstrap-n-strains.html">Bootstrap n antigen samples (n=num antigens in sequenced merge) of the seq-t4 and seq-t5</a>
  <li><a href="investigations/seasons/show-bootstrap-variation-with-procrustes.html">Show bootstrap variation with procrustes</a>
  <li><a href="investigations/seasons/show-bootstrap-variation-with-blobs.html">Show bootstrap variation with blobs</a> (<a href="investigations/seasons/which-strains-have-the-most-slop.html">blob area calculations</a>)
  <li><a href="investigations/seasons/bootstraps.html">Bootstraps with the same number of strains per season (also in the seasons section above)</a>
  <li><a href="investigations/shape/bootstrap-titers.html">Bootstrap titers</a>
  <li><a href="investigations/merge-hi-tables/seq-t8-bootstrap.html">seq-t8 bootstrap (ag sr and titers)</a>
  <li><a href="investigations/merge-hi-tables/seq-t8a-bootstrap.html">seq-t8a bootstrap sera looking for the sera responsible for the kink</a>
  <li><a href="investigations/merge-hi-tables/seq-t9a-bootstrap.html">seq-t9a bootstraps (also listed in prep for ms)</a>
  <li><a href="investigations/merge-hi-tables/seq-t9a-mod15-bootstraps.html">seq-t9a mod15 bootstraps (also listed in merge-hi-tables section)</a>
  <li><a href="investigations/merge-hi-tables/seq-t9a-mod27-bootstraps.html">seq-t9a mod27 bootstraps (also listed in merge-hi-tables section)</a>
  </UL>     


<H1> Consensus trees</H1>
  <UL>
  <li><a href="investigations/consensus-trees/multiple-restarts.html">100 random starts of seq-t5</a>
  <li><a href="investigations/slop/bootstraps.html">Boostraps</a>
  </UL>


<H1> Constant-force locii</H1>
  <UL>
  <li><a href="investigations/slop/initial.html">Slop</a>
  <li><a href="investigations/slop/multiple-optima.html">Slop -- using the tool to look for multiple optima</a>
  <li><a href="investigations/slop/error-and-connection-lines.html">Slop -- error and connection lines</a>
  </UL>


<H1> Square kernels</H1>
  <UL>
  <li><a href="investigations/square-kernels/initial.html">Initial generation of a set of square kernels</a>
  <li><a href="investigations/square-kernels/procrustes-onto-existing-as-check.html">Ues the above kernels as a check on the sequenced-strains take 4 map</a>
  </UL>


<H1> Different labs</H1>
  <UL>
  <li><a href="investigations/rivm-eur/seq-and-all-procrustes.html">RIVM and EUR H3N2</a>
  <li><a href="investigations/merge-hi-tables/viro-merge-2002-10.html">Take 2 viro merge, 2002-10</a>
  </UL>


<H1>Validation</H1>
  <UL>
  <li><a href="investigations/validation/randomize-points.html">Randomize points in final configuration</a>
  <li>The constant force locii stuff (in its own section above)
  </UL>


<H1>Epidemiology</H1>
  <UL>
  <li><a href="investigations/epidemiology/initial.html">Initial look at the epi data</a>
  <li><a href="investigations/epidemiology/eiss-initial.html">Initial look at the EISS data</a>
  <li><a href="investigations/epidemiology/cluster-jump-after-low-incidence.html">Does a low incidence year induce a cluster jump?</a>
  <li><a href="investigations/epidemiology/genetic-map-for-bryan.html">Genetic map for Byran</a>
  </UL>


<H1> Ferret reinfection</H1>
  <UL>
  <li><a href="investigations/ferret-reinfection/ferret-reinfection.html">Ferret reinfection experiment analysis</a>
  <li><a href="investigations/ferret-reinfection/initial-ledger-process.html">Initial ledger processing</a>
  <p>
  <li><a href="investigations/ferret-reinfection/cleaned-ferret-data.html">Cleaned ferret data (page made 2010-07-19, data cleaned at time of initial analyses above)</a>
  <p>
  <li><a href="investigations/ferret-reinfection/gen-parameters.html">Setting up the simulator (java) run</a>
  <p>
  <li><a href="investigations/ferret-reinfection/pre-post-slopes.html">Pre-Post slopes</a>
  <li><a href="investigations/ferret-reinfection/pre-post-slopes-summary.html">Pre-Post slopes summary</a>
  <li><a href="investigations/ferret-reinfection/map-take-2.html">Antigenic map (take 2, March 2002)</a>
  <li><a href="investigations/ferret-reinfection/larger-75-b-table.html">Larger map that includes the reinfection strains</a>
  <li><a href="investigations/ferret-reinfection/landscapes.html">Landscapes</a>
  <li><a href="investigations/ferret-reinfection/infecting-far-away.html">Infecting far away</a>
  <li><a href="investigations/ferret-reinfection/simple-model.html">Simple model</a>
  <li><a href="investigations/ferret-reinfection/simple-model-applied-to-ferret-data.html">Simple model applied to ferret data</a>
  <li><a href="data/cdc-b/cdc-b.html">CDC B tables</a>
  <li><a href="investigations/ferret-reinfection/slope-intercept-correlation.html">Intercept slope correlation</a>
  <p>
  <li><a href="investigations/ferret-reinfection/pnas-simulator-gridware/collate-data.html">All experiments data (Q1 2003)</a>
  <li><a href="investigations/ferret-reinfection/manuscript-draft.html">Manuscript draft</a>
  <p>
  <li><a href="investigations/ferret-reinfection/sequences.html">Sequences</a>
  </UL>


<H1> Sy-Fu vaccine trial</H1>
  <UL>
  <li><a href="investigations/sy-fu-vaccine-trial/human-and-ferret-missmatch.html">The missmatch data</a>
  <li><a href="investigations/sy-fu-vaccine-trial/strains-for-antibody-landscapes.html">Strains for antibody landscapes</a>
  <li><a href="investigations/merge-hi-tables/solvay-sera-titrations.html">Strains for Solvay titrations, take 2</a>
  <li><a href="investigations/merge-hi-tables/solvay-sera-titrations-right-side-sera.html">Strains for Solvay titrations -- choosing a right-side serum</a>
  <li><a href="investigations/merge-hi-tables/solvay-sera-titrations-alt-to-nl32503.html">Strains for Solvay titrations -- alternate to nl/325/03 (2004-08-12)</a>
  <p>
  <li><a href="investigations/sy-fu-vaccine-trial/data-processing.html">Raw data processing</a>
  <li><a href="investigations/sy-fu-vaccine-trial/data-processing-2.html">Initial analysis (Aug 2005)</a>
  <li><a href="investigations/sy-fu-vaccine-trial/compare-ferret-titers.html">Compare raw ferret titers: ferret concomitant, 2005, and 2009 35-year maps (2010-11-26)</a>
  <li><a href="investigations/sy-fu-vaccine-trial/fit-ferret-model.html">Test fit of ferret model to human SY-FU vaccine trial data (2008-03-17)</a>
  <li><a href="investigations/sy-fu-vaccine-trial/find-inconsistent-human-points.html">Find inconsistent points to human sera (2008-03-23)</a>
  <li><a href="investigations/sy-fu-vaccine-trial/fit-ferret-model-serology-map.html">Test fit of ferret model to human SY-FU vaccine trial data as above, but to a base map made from the control ferret titers in the human serology experiment (2008-03-30)</a>
  <li><a href="investigations/sy-fu-vaccine-trial/find-serum-point-for-serology-cones.html">Find best serum point for serology cones (2008-03-30)</a>
  <li><a href="investigations/sy-fu-vaccine-trial/direct-human-maps.html">Direct human maps from the Serology (suggestion from Ron) (2008-05-07)</a>
  <p>
  <li><a href="investigations/sy-fu-vaccine-trial/2009-thoughts.html">Some 2009 thoughts (2009-04-22)</a>
  <p>
  <li><a href="investigations/sy-fu-vaccine-trial/1d.html">1D maps and landscapes (2010-10-17)</a>
  <li><a href="investigations/sy-fu-vaccine-trial/residuals.html">Plot pre-post residuals on the 1d landscapes (Sarah & Derek 2010-11-11)</a>
  <li><a href="investigations/sy-fu-vaccine-trial/1d-with-hk68.html">Same 1D maps and landscapes as 2 links above, but with HK68 ag added, and look how HK68 titers relate to age (2010-11-20)</a>
  <p>
  <li><a href="investigations/sy-fu-vaccine-trial/2011-base-map.html">July 2011 base map (2011-07-21)</a>
  <p>
  <li><a href="investigations/oz-landscapes/index.html">Oz lanscapes base map (2011-10-24)</a>
  </UL>


<H1> Strain-selection meeting SH 2004 </H1>
  <UL>
  <li><a href="investigations/strain-selection-meeting/plan.html">Plan</a>
  <li><a href="investigations/strain-selection-meeting/TODO.html">Random TODO thoughts</a>
  <p>
  <li><h3>Infrastructure</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/scripts/individual-table/single-table-script.html">First cut at script for each table</a>
  <li><a href="investigations/strain-selection-meeting/database/scripts/individual-table/single-table-script-take-2.html">Second cut at script for each table</a>
  <li><a href="investigations/strain-selection-meeting/database/scripts/individual-table/single-table-script-take-3.html">Third cut at script for each table</a>  <p>
  <li><h3>CDC</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/cdc/raw-html/index.html">CDC tables in HTML format (to save having to open excel)</a>
  <li><a href="investigations/strain-selection-meeting/database/cdc/index.html">Initial batch processing of CDC HI tables</a>
  <li><a href="investigations/strain-selection-meeting/database/cdc/investigations/reference-antigen-map/reference-antigen-comparison-index.html">Initial reference antigen comparison</a>
  <li><a href="investigations/strain-selection-meeting/database/cdc/investigations/reference-antigen-map/reference-antigen-comparison-overlay-merge.html">Overlay merge of multiple tables</a>
  <p>
  <li><h3>NIMR</h3></a>
  <li><a href="investigations/strain-selection-meeting/map-with-vaccine-strains.html">Map with vaccine strains marked</a>
  <li><a href="investigations/strain-selection-meeting/database/nimr/raw-html/index.html">NIMR tables in HTML format (to save having to open excel)</a>
  <li><a href="investigations/strain-selection-meeting/database/nimr/index.html">Initial batch processing of NIMR HI tables</a>
  <li><a href="investigations/strain-selection-meeting/database/nimr/raw-data/nimr-sequences.html">NIMR sequences</a>
  <li><a href="investigations/strain-selection-meeting/database/nimr/investigations/reference-antigen-map/index.html">Reference antigen map</a>  
  <p>
  <li><h3>NIID</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/niid/raw-data/index.html">will eventually link to the japanese data</a>
  <li><a href="investigations/strain-selection-meeting/database/niid/investigations/reference-antigen-map/index.html">Reference antigen map</a>  
  <li><a href="investigations/strain-selection-meeting/database/niid/runs/20040909a-ref-ag-merge/index.html">Runs of individual tables merged with the ref ag map</a>  
  <p>
  <li><h3>CSL</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/melb/raw-data/csl.zip">Original CSL HI zip</a>
  <li><a href="investigations/strain-selection-meeting/database/melb/raw-data/HI-as-html-index.html">Original HI excel sheets as html</a>
  <p>
  <li><h3>Automated all together</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/combined/reference-maps/pc-among-labs/20040915/index.html">Reference map comparisons among labs</a>  
  <li><a href="investigations/strain-selection-meeting/database/combined/reference-maps/pc-among-labs/20040915/index2.html">Reference map comparisons among labs (different coloring)</a>  
  <li><a href="investigations/strain-selection-meeting/database/combined/reference-maps/composite/index.html">Single reference map for all labs</a>  

  <p> 
  <li><a href="investigations/strain-selection-meeting/database/cdc/investigations/reference-antigen-map/20040820/index.html">CDC reference map, and individual tables merged with reference map</a>  
  <li><a href="investigations/strain-selection-meeting/database/niid/investigations/reference-antigen-map/20040910/index.html">NIID reference map, and individual tables merged with reference map</a>  
  <li><a href="investigations/strain-selection-meeting/database/nimr/investigations/reference-antigen-map/20040910/index.html">NIMR reference map, and individual tables merged with reference map</a>  
  <li><a href="investigations/strain-selection-meeting/database/melb/investigations/reference-antigen-map/20040912/index.html">CSL reference map, and individual tables merged with reference map</a>  (and <a href="investigations/strain-selection-meeting/database/melb/investigations/reference-antigen-map/20040910/index.html">before name grouping</a>).
  <p>
  <li><a href="investigations/strain-selection-meeting/database/cdc/investigations/reference-antigen-map/reference-antigen-comparison-overlay-merge.html">CDC overlay merge (also above, this was done previously)</a>
  <li><a href="investigations/strain-selection-meeting/database/niid/investigations/overlay-merge/20040910-ref-ag-merge/index.html">NIID overlay merge</a>
  <li><a href="investigations/strain-selection-meeting/database/nimr/investigations/overlay-merge/20040910-ref-ag-merge/index.html">NIMR overlay merge</a>
  <li><a href="investigations/strain-selection-meeting/database/melb/investigations/overlay-merge/20040912-ref-ag-merge/index.html">CSL overlay merge</a>
  <p>
  <li><h3>Sequence</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/combined/3-lab-genetic-map/index.html">CSL+NIID+NIMR+EMC genetic map</a>  
  </UL>


<H1> Strain-selection meeting NH 2005 </H1>
  <UL>
  <li><h3>Misc</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/misc/2005-nh-2004-sh-data-consistency-checks.html">Consistency checks on 2004 SH and 2005 NH data</a>  
  <p>
  <li><h3>CDC</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/cdc/investigations/reference-antigen-map/20041229/index.html">CDC reference map, and individual tables merged with reference map</a>  
  <li><a href="investigations/strain-selection-meeting/database/cdc/investigations/reference-antigen-map/20041230/index.html">CDC reference map, and individual tables merged with reference map (Take 2, this time exlude the previously excluded tables, and the new guinea pig tables)</a>  
  <li><a href="investigations/strain-selection-meeting/database/cdc/investigations/reference-antigen-map/20041230-20040820-grouped-names/index.html">CDC reference map, and individual tables merged with reference map (Take 3, with name merging from the 2004 SH reference ag map)</a>  
  <li><a href="investigations/strain-selection-meeting/database/cdc/investigations/reference-antigen-map/20041230-20040820-grouped-names-100-runs/index.html">CDC reference map, and individual tables merged with reference map (Take 4, as take 3 but with 100 runs)</a>  
  <li><a href="investigations/strain-selection-meeting/database/cdc/investigations/reference-antigen-map/20041231-grouped-names/index.html">CDC reference map, and individual tables merged with reference map (Take 5, 20040820 name grouping with 20041230 name groups added (100 runs))</a>
  <li><a href="investigations/strain-selection-meeting/database/cdc/runs/20041231-grouped-names/runs-index.html">CDC 20041231-grouped-names</a>  
  <li><a href="investigations/strain-selection-meeting/database/cdc/runs/20041231-grouped-names-ref-ag-merge/runs-index.html">CDC 20041231-grouped-names-ref-ag-merge</a>  
  <p>
  <li><a href="investigations/strain-selection-meeting/database/cdc/investigations/overlay-merge/20041231-grouped-names-ref-ag-merge/index.html">Overlay merge: CDC 20041231-grouped-names-ref-ag-merge</a>  
  <p>
  <li><a href="investigations/strain-selection-meeting/database/cdc/investigations/kr77002/index.html">KR/770/02 serum</a>  
  <p>
  <li><a href="investigations/strain-selection-meeting/database/cdc/investigations/genetic-maps/20041228/initial.html">Genetic maps</a>  
  <p>
  <li><h3>MEBL</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/melb/investigations/reference-antigen-map/20041231-20040910-grouped-names/index.html">MELB reference map, and individual tables merged with reference map (with name grouping from the 2004SH name groups)</a>  
  <li><a href="investigations/strain-selection-meeting/database/melb/investigations/reference-antigen-map/20050101-grouped-names/index.html">MELB reference map, and individual tables merged with reference map (with name grouping post the post 2004SH name grouping)</a>  
  <li><a href="investigations/strain-selection-meeting/database/melb/runs/20050101-grouped-names/runs-index.html">MELB 20050101-grouped-names</a>  
  <li><a href="investigations/strain-selection-meeting/database/melb/runs/20050101-grouped-names-ref-ag-merge/runs-index.html">MELB 20050101-grouped-names-ref-ag-merge</a>  
  <p>
  <li><a href="investigations/strain-selection-meeting/database/melb/investigations/overlay-merge/20050101-grouped-names-ref-ag-merge/index.html">Overlay merge: (2nd half) MELB 20050101-grouped-names-ref-ag-merge</a>  
  <p>
  <li><h3>NIMR</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/nimr/runs/20050103-incremental-tables-from-20040910-ref-ag-merge/runs-index.html">NIMR 20050103-ref-ag-merge</a>
  <li><a href="investigations/strain-selection-meeting/database/nimr/investigations/overlay-merge/20050103-incremental-tables-from-20040910-ref-ag-merge/index.html">Overlay merge: NIMR</a>  
  <p>
  <li><a href="investigations/strain-selection-meeting/database/nimr/investigations/table-grouping/index.html">Merging tables with the same reference set, and which were made on the same date</a>  
  <li><a href="investigations/strain-selection-meeting/database/nimr/runs/20050120-date-grouped/runs-index.html">Grouped tables runs</a>  
  <li><a href="investigations/strain-selection-meeting/database/nimr/runs/20050120-individual-tables/runs-index.html">Individual tables runs (for comparison with above)</a>  
  <li><a href="investigations/strain-selection-meeting/database/nimr/investigations/table-grouping/merge-diagnostics/index.html">Procrustes diagnostics of the grouped tables</a>  
  <p>
  <li><a href="investigations/strain-selection-meeting/database/nimr/investigations/table-grouping/groups-in-same-table.html">Initial look at 20040926-Dakar, as there are two groups, are they real?</a>  
  <li><a href="investigations/strain-selection-meeting/database/nimr/investigations/table-grouping/israel-switz.html">Initial look at 20040130-israel-switz, as there are three potential groups, are they real?</a>  
  <p>
  <li><a href="investigations/strain-selection-meeting/database/nimr/investigations/sera-separating/index.html">Assume NIMR sera are different before proven the same</a>  
  <li><a href="investigations/strain-selection-meeting/database/nimr/investigations/reference-antigen-map/20050121-separated-sera/index.html">NIMR reference map with all sera separated</a>  
  <p>
  <li><h3>NIID</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/niid/runs/20050103-incremental-tables-from-20040910-ref-ag-merge/runs-index.html">NIID 20050103-ref-ag-merge</a>
  <li><a href="investigations/strain-selection-meeting/database/niid/investigations/overlay-merge/20050103-incremental-tables-from-20040910-ref-ag-merge/index.html">Overlay merge: NIID</a>  
  <p>
  <li><h3>EMC</h3></a>
  <li><a href="investigations/merge-hi-tables/emc-2005-update.html">2004/2005 update into previously updated Science ms map</a>  
  <p>
  <li><h3>Sequence</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/cdc/investigations/genetic-maps/20041228/initial.html">Initial runs on CDC sequences (Sept 2002 to Nov 2004)</a>  
  <p>
  <li><h3>Misc</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/cdc/investigations/overlay-merge/20041231-grouped-names-ref-ag-merge/display-methods.html">Display methods</a>  
  </UL>


<H1> Strain-selection meeting NH 2005 2nd teleconference</H1>
  <UL>
  <li><h3>CDC</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/flows/2005-NH-2nd-teleconference-cdc.html">CDC flow</a>  
  <li><a href="investigations/strain-selection-meeting/database/cdc/runs/20050128-grouped-names/runs-index.html">CDC 20050128 tables, grouped names (without lab master reference merge)</a>    
  <li><a href="investigations/strain-selection-meeting/database/cdc/investigations/reference-antigen-map/20050128/index.html">CDC master reference map, and individual tables merged with master reference map, 20050128 (no name grouping)</a>  
  <li><a href="investigations/strain-selection-meeting/database/cdc/investigations/reference-antigen-map/20050128-grouped-names/index.html">CDC master reference map, and individual tables merged with master reference map, 20050128-grouped-names</a>  
  <p>
  <li><a href="investigations/strain-selection-meeting/database/cdc/runs/20050128-grouped-names/runs-index.html">CDC 20050205-grouped-names which adds one table incrementally to 20050128</a>  
  <li><a href="investigations/strain-selection-meeting/database/cdc/investigations/reference-antigen-map/20050128-grouped-names/index.html">CDC 20050205-grouped-names-ref-ag-merge which adds one table incrementally to 20050128</a>  
  <p>
  <li><a href="investigations/strain-selection-meeting/database/cdc/investigations/overlay-merge/20050128-grouped-names-ref-ag-merge/index.html">CDC overlay merge 20050128-grouped-names</a>  
  <p>
  <li><h3>MEBL</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/melb/runs/20050128-grouped-names/runs-index.html">MELB 20050128 tables, grouped names (without lab master reference merge)</a>  
  <li><a href="investigations/strain-selection-meeting/database/melb/investigations/reference-antigen-map/20050128-with-grouped-names-from-20050101/index.html">MELB master reference map, and individual tables merged with master reference map, 20050128-with-grouped-names-from-20050101</a>  
  <li><a href="investigations/strain-selection-meeting/database/melb/investigations/reference-antigen-map/20050128-grouped-names/index.html">MELB master reference map, and individual tables merged with master reference map, 20050128-grouped-names</a>  
  <p>
  <li><h3>NIMR</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/nimr/runs/20050128/runs-index.html">NIMR 20050128 tables (without lab master reference merge)  Has some bad serum names from our table reading</a>  
  <li><a href="investigations/strain-selection-meeting/database/nimr/investigations/reference-antigen-map/20050128/index.html">NIMR 20050128 (with lab master reference merge)  Has some bad serum names from our table reading</a>  
  <p>
  <li><a href="investigations/strain-selection-meeting/database/nimr/runs/20050131-with-ferret-numbers/runs-index.html">NIMR 20050131 tables (without lab master reference merge)  Fixes sera names from 20050128 (with ferret numbers in name)</a>  
  <li><a href="investigations/strain-selection-meeting/database/nimr/investigations/reference-antigen-map/20050131-with-ferret-numbers/index.html">NIMR 20050131 (with lab master reference merge) Fixes sera names from 20050128 (with ferret numbers in name) (DOES NOT LINK CORRECTLY)</a>  
  <p>
  <li><a href="investigations/strain-selection-meeting/database/nimr/runs/20050131/runs-index.html">NIMR 20050131 tables (without lab master reference merge)  Fixes sera names from 20050128, (no ferret numbers in name)</a>  
  <li><a href="investigations/strain-selection-meeting/database/nimr/investigations/reference-antigen-map/20050131/index.html">NIMR 20050131 (with lab master reference merge) Fixes sera names from 20050128 (no ferret numbers in name)</a>  
  <p>
  <li><h3>NIID</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/niid/runs/20050131/runs-index.html">NIID 20050128 tables (without lab master reference merge)</a>  
  <li><a href="investigations/strain-selection-meeting/database/niid/investigations/reference-antigen-map/20050128/index.html">NIID 20050128 (with lab master reference merge)</a>  
  <p>
  <li><h3>Sequence</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/combined/investigations/genetic-maps/20050204/index.html">Combined Genetic map, and timeseries</a>  
  <li><a href="investigations/strain-selection-meeting/database/combined/investigations/genetic-maps/20050204/unique-aa-routine-diagnostics/index.html">Combined Genetic map, routine diagnostics</a>  

  <p>
  <li><h3>Antigenic effect of substitutions</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/cdc/investigations/overlay-merge/20050128-grouped-names-ref-ag-merge/20050128-relaxed-20050131-seq/index.html">CDC antigenic effect of substitutions</a>  
  <li><a href="investigations/strain-selection-meeting/database/melb/investigations/overlay-merge/20050128-grouped-names-ref-ag-merge/20050128-shape-20050131-seq/index.html">MEBL antigenic effect of substitutions</a>  
  <p>
  <li><h3>"-like" analyses</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/combined/investigations/like-timeseries/20050203/index.html">-like analyses (2nd teleconference data)</a>  
  <p>
  <li><h3>Misc</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/misc/2005-NH-notes.html">Notes for meeting (such as 2 groups of CDC WE sera)</a>  
  </UL>


<H1> Strain-selection meeting NH 2005 consultation</H1>
  <UL>
  <li><h3>CDC</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/cdc/runs/20050128-grouped-names/runs-index.html">CDC 20050205-grouped-names which adds one table incrementally to 20050128</a>  
  <li><a href="investigations/strain-selection-meeting/database/cdc/investigations/reference-antigen-map/20050128-grouped-names/index.html">CDC 20050205-grouped-names-ref-ag-merge which adds one table incrementally to 20050128</a>  
  <p>
  <li><a href="investigations/strain-selection-meeting/database/cdc/investigations/overlay-merge/20050128-grouped-names-ref-ag-merge/index.html">CDC overlay merge 20050128-grouped-names</a>  
  <p>
  <li><h3>MEBL</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/melb/runs/20050128-grouped-names/runs-index.html">MELB 20050205-grouped-names (which adds two tables incrementally to 20050128-grouped-names)</a>  
  <li><a href="investigations/strain-selection-meeting/database/melb/investigations/reference-antigen-map/20050128-grouped-names/index.html">MELB 20050205-grouped-names (which adds two tables incrementally to 20050128-grouped-names) ref-ag-merge</a>  
  <p>
  <li><h3>NIMR</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/nimr/runs/20050131/runs-index.html">NIMR 20050205 (incremental from 20050131)</a>  
  <li><a href="investigations/strain-selection-meeting/database/nimr/investigations/reference-antigen-map/20050131/index.html">NIMR 20050205 (incremental from 20050131) (with lab master reference merge)</a>  
  <p>
  <li><h3>NIID</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/niid/runs/20050128/runs-index.html">NIID 20050128 tables (without lab master reference merge)</a>  
  <li><a href="investigations/strain-selection-meeting/database/niid/investigations/reference-antigen-map/20050128/index.html">NIID 20050128 (with lab master reference merge)</a>  
  <p>
  <li><h3>Sequence</h3></a>
  <p>
  <li><h3>Antigenic effect of substitutions</h3></a>
  <p>
  <li><h3>"-like" analyses</h3></a>
  <p>
  <li><h3>Misc</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/combined/investigations/sp-serum/index.html">SP/37/04 serum</a>  
  </UL>


<H1> Strain-selection meeting SH 2005 2nd teleconference</H1>
  <UL>
  <li><h3>CDC</h3></a>
  <li>(<a href="investigations/strain-selection-meeting/database/cdc/investigations/reference-antigen-map/20050713-20050205-grouped-names/index.html">Reference antigen map 20050713-20050205-grouped-names (start with the 20050205 name grouping to see what to do with the new strains since then)</a>  )
  <p>
  <li><a href="investigations/strain-selection-meeting/database/cdc/runs/20050713-grouped-names/runs-index.html">20050831 individual tables (using 20050713-grouped-names) (without master ref ag merge)</a>  
  <li><a href="investigations/strain-selection-meeting/database/cdc/investigations/reference-antigen-map/20050713-grouped-names/index.html">20050831 individual tables (with ref ag merge) (20050713-grouped-names and 20050713-ref-ag-merge)</a>  
  <p>
  <li><a href="investigations/strain-selection-meeting/database/cdc/investigations/overlay-merge/20050713-grouped-names-ref-ag-merge/index.html">CDC overlay merge 20050713-grouped-names (lots of other workings in there too, looking at hemisphering)</a>
  <li>(<a href="investigations/strain-selection-meeting/database/cdc/investigations/reference-antigen-map/20050713-grouped-names-list-merge/index.html">Reference antigen map 20050713-grouped-names-list-merge -- seeing if accounting for repeated titrations helps with hemisphereing seeing in 20050311 (and related tables)</a>)
  <li>(<a href="investigations/strain-selection-meeting/database/cdc/investigations/reference-antigen-map/20050713-grouped-names-list-merge-all/index.html">Same as above, this time with all of the ref ag merge, and 25 runs</a>)
  <p>
  <li><h3>MEBL</h3></a>
  <li>(<a href="investigations/strain-selection-meeting/database/melb/investigations/reference-antigen-map/20050718-20050205-grouped-names/index.html">Reference antigen map 20050718 with groupings from 20050205 (to look at any name grouping)</a>)
  <p>
  <li><a href="investigations/strain-selection-meeting/database/melb/runs/20050718-grouped-names/runs-index.html">20050901 individual tables (using 20050718-grouped-names)</a>  
  <li><a href="investigations/strain-selection-meeting/database/melb/investigations/reference-antigen-map/20050718-grouped-names/index.html">20050901 individual tables with ref ag merge (using 20050718 name grouping and ref ag merge)</a>  
  <p>
  <li><a href="investigations/strain-selection-meeting/database/melb/investigations/overlay-merge/20050718-grouped-names-ref-ag-merge/index.html">Overlay merge 20050901</a>
  <li><h3>NIID</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/niid/runs/20050717/runs-index.html">20050831 individual tables</a>  
  <li><a href="investigations/strain-selection-meeting/database/niid/investigations/reference-antigen-map/20050717/index.html">20050831 individual tables with ref ag merge (using 20050717 ref ag merge)</a> 
  <p>
  <li><a href="investigations/strain-selection-meeting/database/niid/investigations/overlay-merge/20050717-ref-ag-merge/index.html">Overlay merge 20050831</a>
  <li><h3>NIMR</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/nimr/runs/20050717/runs-index.html">20050717 individual tables</a>  
  <li><a href="investigations/strain-selection-meeting/database/nimr/investigations/reference-antigen-map/20050717/index.html">20050717 individual tables with ref ag merge</a>  
  <p>
  <li><a href="investigations/strain-selection-meeting/database/nimr/investigations/overlay-merge/20050717-ref-ag-merge/index.html">Overlay merge 20050901a</a>
  <li><h3>Sequence</h3></a>
  <p>
  <li><h3>Antigenic effect of substitutions</h3></a>
  <p>
  <li><h3>"-like" analyses</h3></a>
  <p>
  <li><h3>Misc</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/flows/2005-SH-teleconference-2.html">Generating the above data</a>
  <li><h3>Report</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/reports/2005-SH-TC2.pdf">2nd teleconference report</a>
  <li><a href="investigations/strain-selection-meeting/database/reports/2005-SH-TC2.pngs.zip">2nd teleconference report pngs</a>
  </UL>


<H1> Strain-selection meeting 2006 NH 1st teleconference</H1>
  <UL>
  <li><h3>CDC</h3></a>
  <li>(<a href="investigations/strain-selection-meeting/database/cdc/investigations/reference-antigen-map/20051216-20050713-grouped-names/index.html">Master reference map 20051216-20050713-grouped-names (interim map used to group names for the master reference map below)</a>)
  <li><a href="investigations/strain-selection-meeting/database/cdc/investigations/reference-antigen-map/20051216-grouped-names/index.html">Master reference map 20051216-grouped-names</a>
  <p>
  <li><a href="investigations/strain-selection-meeting/database/cdc/runs/20051216-grouped-names/runs-index.html">Individual tables (using 20051216-grouped-names)</a>  
  <li><a href="investigations/strain-selection-meeting/database/cdc/investigations/reference-antigen-map/20051216-grouped-names/index.html">Individual tables merged with the master reference map (using 20051216-grouped-names) (same as master-reference-map link above)</a>
  <p>
  <li><a href="investigations/strain-selection-meeting/database/cdc/investigations/overlay-merge/20051216-grouped-names-ref-ag-merge/index.html">Overlay merge (messy, lots of other workings in there too)</a>
  <p>
  <li><a href="investigations/strain-selection-meeting/database/cdc/investigations/reference-antigen-map/20051216-grouped-names/further-investigations/with-alan.html">Take a look at table 20051103 as a representative of the meta-stability in the latest CDC tables (with Alan)</a>
  <li><h3>MEBL</h3></a>
  <li>(<a href="investigations/strain-selection-meeting/database/melb/investigations/reference-antigen-map/20051216-20050718-grouped-names/index.html">Reference antigen map 20051216-20050713-grouped-names (intermim map used to group names for the master reference map below)</a>)
  <p>
  <li><a href="investigations/strain-selection-meeting/database/melb/runs/20051216-grouped-names/runs-index.html">Individual tables (using 20051216-grouped-names)</a>  
  <li><a href="investigations/strain-selection-meeting/database/melb/investigations/reference-antigen-map/20051216-grouped-names/index.html">Individual tables merged with the master reference map (using 20051216-grouped-names)</a>
  <p>
  <li><a href="investigations/strain-selection-meeting/database/melb/investigations/overlay-merge/20051216-grouped-names-ref-ag-merge/index.html">Overlay merge</a>
  <li><h3>NIID</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/niid/runs/20051216/runs-index.html">Individual tables</a>  
  <li><a href="investigations/strain-selection-meeting/database/niid/investigations/reference-antigen-map/20051216/index.html">Individual tables merged with the master reference map (using 20051216-grouped-names)</a>
  <p>
  <li><a href="investigations/strain-selection-meeting/database/niid/investigations/overlay-merge/20051216-ref-ag-merge/index.html">Overlay merge</a>
  <li><h3>NIMR</h3></a>
  <li>No new data
  <p>
  <li><h3>Sequence</h3></a>
  <p>
  <li><h3>Antigenic effect of substitutions</h3></a>
  <p>
  <li><h3>"-like" analyses</h3></a>
  <p>
  <li><h3>Misc</h3></a>
  <li><h3>Report</h3></a>
  </UL>


<H1> Strain-selection meeting 2006 NH 2nd teleconference</H1>
  <UL>
  <li><h3>CDC</h3></a>
  <li>(<a href="investigations/strain-selection-meeting/database/cdc/investigations/reference-antigen-map/20060127-20051216-grouped-names/index.html">Master reference map 20060127-20051216-grouped-names (interim map used to group names for the master reference map below)</a>)
  <li><a href="investigations/strain-selection-meeting/database/cdc/investigations/reference-antigen-map/20060127-grouped-names/index.html">Master reference map 20060127-grouped-names</a>
  <p>
  <li><a href="investigations/strain-selection-meeting/database/cdc/runs/20060127-grouped-names/runs-index.html">Individual tables (using 20060127-grouped-names)</a>  
  <li><a href="investigations/strain-selection-meeting/database/cdc/investigations/reference-antigen-map/20060127-grouped-names/index.html">Individual tables merged with the master reference map (using 20060127-grouped-names) (same as master-reference-map link above)</a>
  <p>
  <li><a href="investigations/strain-selection-meeting/database/cdc/investigations/overlay-merge/20060127-grouped-names-ref-ag-merge/index.html">Overlay merge</a>
  <li><h3>MEBL</h3></a>
  <li>The 2 new tables (receipt date 20060130) since the 1st teleconference have been added to the Melb 1st teleconference individual and ref-ag-merge table pages above (overlay merge not yet updated)
  <li><h3>NIID</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/niid/runs/20060130/runs-index.html">Individual tables</a>  
  <li><a href="investigations/strain-selection-meeting/database/niid/investigations/reference-antigen-map/20060130/index.html">Individual tables merged with the master reference map</a>
  <p>
  <li><a href="investigations/strain-selection-meeting/database/niid/investigations/overlay-merge/20060130-ref-ag-merge/index.html">Overlay merge</a>
  <li><h3>NIMR</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/nimr/runs/20050717/runs-index.html">Individual tables</a>  
  <li><a href="investigations/strain-selection-meeting/database/nimr/investigations/reference-antigen-map/20050717/index.html">Individual tables merged with the master reference map</a>
  <p>
  <li><a href="investigations/strain-selection-meeting/database/nimr/investigations/overlay-merge/20060210-ref-ag-merge/index.html">Overlay merge</a>
  <li><h3>Sequence</h3></a>
  <p>
  <li><h3>Antigenic effect of substitutions</h3></a>
  <p>
  <li><h3>"-like" analyses</h3></a>
  <p>
  <li><h3>Misc</h3></a>
  <li><h3>Report</h3></a>
  </UL>



<H1> Strain-selection meeting 2006 SH 1st teleconference</H1>
  <UL>
  <li><h3>CDC</h3></a>
  <li>(<a href="investigations/strain-selection-meeting/database/cdc/investigations/reference-antigen-map/20060731-20060127-grouped-names/index.html">Master reference map 20060731-20060127-grouped-names (interim map used to group names for the master reference map below)</a>)
  <li><a href="investigations/strain-selection-meeting/database/cdc/investigations/reference-antigen-map/20060731-grouped-names/index.html">Master reference map 20060731-grouped-names</a>
  <p>
  <li><a href="investigations/strain-selection-meeting/database/cdc/runs/20060731-grouped-names/runs-index.html">Individual tables (using 20060731-grouped-names)</a>  
  <li><a href="investigations/strain-selection-meeting/database/cdc/investigations/reference-antigen-map/20060731-grouped-names/index.html">Individual tables merged with the master reference map (using 20060731-grouped-names) (same as master-reference-map link above)</a>
  <p>
  <li><a href="investigations/strain-selection-meeting/database/cdc/investigations/overlay-merge/20060731-grouped-names-ref-ag-merge/index.html">Overlay merge</a>
  <li><h3>MEBL</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/melb/investigations/reference-antigen-map/20060731-grouped-names/index.html">Master reference map 20060731-grouped-names (was made with 20051216 groups, but no additional grouping was needed)</a>
  <p>
  <li><a href="investigations/strain-selection-meeting/database/melb/runs/20060731-grouped-names/runs-index.html">Individual tables (using 20060731-grouped-names)</a>  
  <li><a href="investigations/strain-selection-meeting/database/melb/investigations/reference-antigen-map/20060731-grouped-names/index.html">Individual tables merged with the master reference map (using 20060731-grouped-names) (same as master-reference-map link above)</a>
  <p>
  <li><a href="investigations/strain-selection-meeting/database/melb/investigations/overlay-merge/20060731-grouped-names-ref-ag-merge/index.html">Overlay merge</a>
  <li><h3>NIID</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/niid/investigations/reference-antigen-map/20060731/index.html">Master reference map</a>
  <p>
  <li><a href="investigations/strain-selection-meeting/database/niid/runs/20060731/runs-index.html">Individual tables</a>  
  <li><a href="investigations/strain-selection-meeting/database/niid/investigations/reference-antigen-map/20060731/index.html">Individual tables merged with the master reference map (same as master-reference-map link above)</a>
  <p>
  <li><a href="investigations/strain-selection-meeting/database/niid/investigations/overlay-merge/20060731-ref-ag-merge/index.html">Overlay merge</a>
  <li><h3>NIMR</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/nimr/investigations/reference-antigen-map/20060731/index.html">Master reference map</a>
  <p>
  <li><a href="investigations/strain-selection-meeting/database/nimr/runs/20060731/runs-index.html">Individual tables</a>  
  <li><a href="investigations/strain-selection-meeting/database/nimr/investigations/reference-antigen-map/20060731/index.html">Individual tables merged with the master reference map (same as master-reference-map link above)</a>
  <p>
  <li><a href="investigations/strain-selection-meeting/database/nimr/investigations/overlay-merge/20060731-ref-ag-merge/index.html">Overlay merge</a>
  </UL>



<H1> Strain-selection meeting 2006 SH 2nd teleconference</H1>
  <UL>
  <li><h3>CDC</h3></a>
  <li>3 tables (20060727 20060804-gp 20060811) merged incrementally into the 1st teleconference above
  <li><h3>MEBL</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/melb/investigations/reference-antigen-map/20060908-grouped-names/index.html">Master reference map 20060908-grouped-names (was made with 20051216 groups--no additional grouping was needed)</a>
  <p>
  <li><a href="investigations/strain-selection-meeting/database/melb/runs/20060908-grouped-names/runs-index.html">Individual tables (using 20060908-grouped-names)</a>  
  <li><a href="investigations/strain-selection-meeting/database/melb/investigations/reference-antigen-map/20060908-grouped-names/index.html">Individual tables merged with the master reference map (using 20060908-grouped-names) (same as master-reference-map link above)</a>
  <li><h3>NIID</h3></a>
  <li>No new data since 1st Teleconference
  <li><h3>NIMR</h3></a>
  <li>3 tables (20060808 20060818 20060830) merged incrementally into the 1st teleconference above
  </UL>



<H1> Strain-selection meeting 2006 SH consultation</H1>
  <UL>
  <li><h3>CDC</h3></a>
  <li>No new data since 2nd Teleconference
  <li><h3>MEBL</h3></a>
  <li>3 tables merged incrementally into the 2nd teleconference above
  <li><a href="investigations/strain-selection-meeting/database/melb/investigations/overlay-merge/20060914-grouped-names-ref-ag-merge/index.html">Overlay merge</a>
  <li><h3>NIID</h3></a>
  <li>No new data since 1st Teleconference
  <li><h3>NIMR</h3></a>
  <li>No new data since 2nd Teleconference
  </UL>



<H1> 2006 06 08 EMC</H1>
  <UL>
  <li><h3>EMC</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/emc/runs/20060608/runs-index.html">Routine diagnostics on the individual tables</a>  
  <p>
  <li><a href="investigations/strain-selection-meeting/database/emc/investigations/overlay-merge/20060608/misc-by-hand/all-tables/index.html">Routine diagnostics on a merge of the 3 recent tables (20060213 20060330 and 20060530)</a>  
  <li><a href="investigations/strain-selection-meeting/database/emc/investigations/overlay-merge/20060608/misc-by-hand/some-pcs.html">Diagnosing the 3-table-merge</a>  
  <p>
  <li><a href="investigations/strain-selection-meeting/database/emc/investigations/overlay-merge/20060608/misc-by-hand/two-tables/index.html">Routine diagnostics on a merge of the two 2006 NL tables (20060213 20060530)</a>  
  <li><a href="investigations/strain-selection-meeting/database/emc/investigations/overlay-merge/20060608/misc-by-hand/two-tables/index2.html">Diagnosing the 2-table-merge</a>  
  <p>
  <li><a href="investigations/strain-selection-meeting/database/emc/investigations/overlay-merge/20060608/misc-by-hand/into-science-plus-save/index.html">Into the science-plus save</a>  
  </UL>

<H1> Strain-selection meeting 2007 NH 1st teleconference</H1>
  <UL>
  <li><h3>CDC</h3></a>
  <li><h4>9 tables (3 are gp tables) merged incrementally into the 2006 SH 1st teleconference above  20061211 plus two more on 20070107 </h4> </a>
  <li><a href="investigations/strain-selection-meeting/database/cdc/runs/20060731-grouped-names/runs-index.html">Individual tables (using 20060731-grouped-names)</a>  
  <li><a href="investigations/strain-selection-meeting/database/cdc/investigations/reference-antigen-map/20060731-grouped-names/index.html">Individual tables merged with the master reference map (using 20060731-grouped-names) (same as master-reference-map link above)</a>
  <li><h3>MEBL</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/melb/runs/20060908-grouped-names/runs-index.html">Individual tables (using 20060908-grouped-names) (incremental into 2006 SH pages)</a>  
  <li><a href="investigations/strain-selection-meeting/database/melb/investigations/reference-antigen-map/20060908-grouped-names/index.html">Individual tables merged with the master reference map (using 20060908-grouped-names) (same as master-reference-map link above) (incremental into 2006 SH pages)</a>
  <li><h3>NIID</h3></a>
  <li><h3>NIMR</h3></a>
  </UL>

<H1> Strain-selection meeting 2007 NH 2nd teleconference</H1>
  <UL>
  <li><h3>CDC</h3></a>
  <li><h3>MEBL</h3></a>
  <li><h3>NIID</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/niid/investigations/overlay-merge/20070201-ref-ag-merge/index.html">Overlay merge</a>
  <li><a href="investigations/strain-selection-meeting/database/niid/runs/20060731/runs-index.html">Individual tables (incremental run of 1 table, same as link above)</a>  
  <li><a href="investigations/strain-selection-meeting/database/niid/investigations/reference-antigen-map/20060731/index.html">Individual tables merged with the master reference map (incremental run of 1 table, same as link above)</a>
  <li><h3>NIMR</h3></a>
  </UL>

<H1> Strain-selection meeting 2007 NH consultation</H1>
  <UL>
  <li><h3>CDC</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/cdc/investigations/overlay-merge/20070128-ref-ag-merge/acmacs/index.html">Acmacs stage 0 run all tables together</a>
  <p>
  <li><a href="investigations/strain-selection-meeting/database/cdc/runs/20060731-grouped-names/runs-index.html">Individual tables (using 20060731-grouped-names)</a>  
  <li><a href="investigations/strain-selection-meeting/database/cdc/investigations/reference-antigen-map/20060731-grouped-names/index.html">Individual tables merged with the master reference map (using 20060731-grouped-names) (same as master-reference-map link above)</a>
  <p>
  <li><a href="investigations/strain-selection-meeting/database/cdc/investigations/overlay-merge/20070209-grouped-names-ref-ag-merge/index.html">Overlay merge</a>
  <p>
  <li><a href="investigations/spatial/timeseries/geographic-antigenic-webpages-equirectangular-outline-full-time-ag-maps-20070209/small-2006-01.html">Time series</a>
  <li><a href="investigations/strain-selection-meeting/database/cdc/investigations/overlay-merge/20070226-grouped-names-ref-ag-merge/index.html">Overlay merge for FDA</a>
  <li><a href="investigations/strain-selection-meeting/database/cdc/investigations/overlay-merge/20070402-grouped-names-ref-ag-merge/index.html">Overlay merge for FDA 2</a>
  <li><a href="investigations/strain-selection-meeting/database/cdc/investigations/overlay-merge/20070407-grouped-names-ref-ag-merge/index.html">Overlay merge for FDA 3</a>
  <li><a href="investigations/strain-selection-meeting/database/cdc/investigations/overlay-merge/20070628-grouped-names-ref-ag-merge/index.html">Overlay merge for June CDC report</a>
  <li><a href="https://data.antigenic-cartography.org/eu/notebook/9020-0000-labs/9090-0000-cdc/8030-overlay-merge/2007-0628-grouped-names-ref-ag-merge">2007-06-28 overlay merge (above) diagnostics in acmacs</a>
  <p>
  <li><a href="investigations/spatial/timeseries/geographic-antigenic-webpages-equirectangular-outline-full-time-ag-maps-20070226/small-2006-01.html">Spatial timeseries</a>
  <li><a href="investigations/spatial/timeseries/geographic-antigenic-webpages-equirectangular-outline-full-time-ag-maps-20070226-aa/small-2006-01.html">Same as above but with avidity adjusts</a>
  <li><h3>MEBL</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/melb/investigations/overlay-merge/20070128-ref-ag-merge/acmacs/index.html">Acmacs stage 0 run all tables together</a>
  <p>
  <li><a href="investigations/strain-selection-meeting/database/melb/runs/20060908-grouped-names/runs-index.html">Individual tables (using 20060908-grouped-names) (incremental into 2006 SH pages)</a>  
  <li><a href="investigations/strain-selection-meeting/database/melb/investigations/reference-antigen-map/20060908-grouped-names/index.html">Individual tables merged with the master reference map (using 20060908-grouped-names) (same as master-reference-map link above) (incremental into 2006 SH pages)</a>
  <p>
  <li><a href="investigations/strain-selection-meeting/database/melb/investigations/overlay-merge/20070209-grouped-names-ref-ag-merge/index.html">Overlay merge</a>
  <li><h3>NIID</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/niid/investigations/overlay-merge/20070128-ref-ag-merge/acmacs/index.html">Looking for stains trapped in a local optimum</a>
  <p>
  <li><a href="investigations/strain-selection-meeting/database/niid/runs/20060731/runs-index.html">Individual tables</a>  
  <li><a href="investigations/strain-selection-meeting/database/niid/investigations/reference-antigen-map/20060731/index.html">Individual tables merged with the master reference map (same as master-reference-map link above)</a>
  <p>
  <li><a href="investigations/strain-selection-meeting/database/niid/investigations/overlay-merge/20070209-ref-ag-merge/index.html">Overlay merge</a>
  <li><h3>NIMR</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/nimr/runs/20060731/runs-index.html">Individual tables</a>  
  <li><a href="investigations/strain-selection-meeting/database/nimr/investigations/reference-antigen-map/20060731/index.html">Individual tables merged with the master reference map (same as master-reference-map link above)</a>
  <p>
  <li><a href="investigations/strain-selection-meeting/database/nimr/investigations/overlay-merge/20070209-ref-ag-merge/index.html">Overlay merge</a>
  </UL>

<H1> Strain-selection meeting, Spring 2007, next generation automation</H1>
  <UL>
  <li><a href="investigations/strain-selection-meeting/database/flows/2007-going-incremental.html">Flow</a>  
  <li><h3>CDC</h3></a>
  <li><h3>MEBL</h3></a>
  <li><h3>NIID</h3></a>
  <li><h3>NIMR</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/nimr/runs/20070310/runs-index.html">Individual tables</a>  
  </UL>

<H1> Strain-selection meeting 2007 SH 1st teleconference (and Migrating the strain selection meeting processing from MDS to ACMACS) </H1>
  <UL>
  <li><h3>CDC</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/cdc/investigations/overlay-merge/20070209-grouped-names-ref-ag-merge/merge-without-master-reference-merge/index.html">Overlay merge without referece antigen merge</a>
  <p>
  <li><a href="investigations/merge-hi-tables/cdc-ref-ag-merge/index.html">Hand optimzations of acmacs's cdc 2000 to 200707 ref ag merge</a>
  <li><a href="need-to-find-this-link">Above relaxed by acmacs</a>
  <li><a href="investigations/merge-hi-tables/cdc-ref-ag-merge/2-and-3d-runs.html">2 and 3D runs of cdc ref ag merge in mds 2000 to 200707</a>
  <p>
  <li><a href="https://data.antigenic-cartography.org/eu/notebook/9020-0000-labs/9090-0000-cdc/8020-0000-merge-non-gp/2007-0731-add-tables-incrementally-one-by-one-2007-backward/126m">2003 to present cdc merge pc with cdc the lispmds way from June 28 2007  (2007-02-08)</a>
  <li><a href="https://data.antigenic-cartography.org/eu/notebook/9020-0000-labs/9090-0000-cdc/8020-0000-merge-non-gp/2007-0802-add-tables-incrementally-one-by-one-2007-backward">2007-07-11 backwards incremental merge (really incremental)  (2007-02-08)</a>
  <li><a href="https://data.antigenic-cartography.org/eu/notebook/9020-0000-labs/9090-0000-cdc/8020-0000-merge-non-gp/2007-0809-incremental-adding-forward/001">2007-07-11 forwards one  (2007-02-09)</a>
  <li><a href="https://data.antigenic-cartography.org/eu/notebook/9020-0000-labs/9090-0000-cdc/8020-0000-merge-non-gp/2007-0811-for-ssm-sh-2007-tc1">TC1 non-aa and aa maps for TC1 (2007-08-12)</a>

  <li><h3>MEBL</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/melb/investigations/overlay-merge/20070209-grouped-names-ref-ag-merge/merge-without-master-reference-merge/index.html">Overlay merge without referece antigen merge</a>
  <p>
  <li><a href="https://data.antigenic-cartography.org/eu/notebook/9020-0000-labs/9080-0000-melb/8020-0000-merge/2007-0508-incremental-tests/2007-0521-add-330th-to-hemi-merge-lha">Melb all-tables optimization 2001 to 2007.1 (2007-05-21)</a>
  <li><a href="https://data.antigenic-cartography.org/eu/notebook/9020-0000-labs/9080-0000-melb/8020-0000-merge/2007-0720-add-tables-incrementally-one-by-one-2007-backward/0-bad">Incremental backwards merge from 20070131 (reoptimizing from scratch at each step (by accident))</a>
  <li><a href="https://data.antigenic-cartography.org/eu/notebook/9020-0000-labs/9080-0000-melb/8020-0000-merge/2007-0720-add-tables-incrementally-one-by-one-2007-backward">Incremental backwards merge from 20070131 (really incremental, after error above fixed)</a>
  <li><a href="https://data.antigenic-cartography.org/eu/notebook/9020-0000-labs/9080-0000-melb/8020-0000-merge/2007-0805-reference-antigens-sera-only">Ref ag sr merge, 500 runs (2007-08-05)</a>
  <li><a href="https://data.antigenic-cartography.org/eu/notebook/9020-0000-labs/9080-0000-melb/8020-0000-merge/2007-0807-optimize-from-scratch-2001-20070717-better-columns-adjusts">All tables 2001 to 2007-07-11 with col bases set to one more than any greater-than (2007-08-08)</a>
  <li><a href="https://data.antigenic-cartography.org/eu/notebook/9020-0000-labs/9080-0000-melb/8020-0000-merge/2007-0809-incremental-forward-from-2001-20070717/005">TC1 full map post hemi/local test (post aa is next link.  note, other labs have both non-aa and aa maps on the one page) (2007-08-12)</a>
  <li><a href="https://data.antigenic-cartography.org/eu/notebook/9020-0000-labs/9080-0000-melb/8020-0000-merge/2007-0811-for-ssm-sh-2007-tc1">TC1 full map post low-avidity adjust and relax on all strains (2007-08-12)</a>

  <li><h3>NIID</h3></a>
  <li><a href="https://data.antigenic-cartography.org/eu/notebook/9020-0000-labs/9070-0000-niid/8020-0000-merge/2007-0801-complete-merge-from-scratch">All tables (2007-08-02)</a>
  <li><a href="investigations/strain-selection-meeting/database/niid/investigations/overlay-merge/20070809-ref-ag-merge/index.html">Take 2007 NH map as starting point, add new points according to map above (2007-08-10)</a>
  <li><a href="https://data.antigenic-cartography.org/eu/notebook/9020-0000-labs/9070-0000-niid/8020-0000-merge/2007-0811-for-ssm-sh-2007-tc1">TC1 non-aa and aa maps for TC1 (2007-08-12)</a>

  <li><h3>NIMR</h3></a>
  <li><a href="https://data.antigenic-cartography.org/eu/notebook/9020-0000-labs/9060-0000-nimr/8020-0000-merge/2007-0802-add-tables-incrementally-one-by-one-2007-backward">First reverse incremental merge (2007-08-02)</a>
  <li><a href="https://data.antigenic-cartography.org/eu/notebook/9020-0000-labs/9060-0000-nimr/8020-0000-merge/2007-0803-add-tables-incrementally-full-optimize-2007-backward">First reverse incremental merge (full opt at each step) (2007-08-02)</a>
  <li><a href="investigations/strain-selection-meeting/database/nimr/investigations/overlay-merge/20070809-ref-ag-merge/when-does-nimr-go-bad.html">Summary of the above in one page to find where NIMR starts to go bad (2007-08-11)</a>
  <li><a href="https://data.antigenic-cartography.org/eu/notebook/9020-0000-labs/9060-0000-nimr/8020-0000-merge/2007-0811-for-ssm-sh-2007-tc1">TC1 non-aa and aa maps for TC1 (2007-08-12)</a>

  <li><h3>Misc</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/niid/investigations/overlay-merge/20070809-ref-ag-merge/avdity-test-timings.html">Avidity test timings (2007-08-12)</a>
  </UL>


<H1> Strain-selection meeting 2007 SH 2nd teleconference & SSM (and continuing migrating the strain selection meeting processing from LISPMDS to ACMACS) </H1>
  <UL>
  <li><h3>CDC</h3></a>
  <li><a href="https://notebooks.antigenic-cartography.org/eu/notebook/?page_id=0709152">CDC tc2-ssm (i missing add one missing table from tc1, then full hemi/local and refag+2006.5onwards low avidity test) (2007-09-15)</a>


  <li><h3>MEBL</h3></a>
  <li><a href="https://notebooks.antigenic-cartography.org/eu/notebook/?page_id=07071408">MELB incremental mege from 2007-SH TC1 to 2007 SH TC2 (2007-09-07 approx)</a>
  <p>
  <li><a href="investigations/strain-selection-meeting/database/combined/investigations/avidity-test-comparisons/2007-SH-tc2-with-tc1/melb/20070822-global-optimum-check/index.html">Melb 20070822ab kink, save manipulation for Eu (2007-09-15)</a>

  <li><h3>NIID</h3></a>
  <li><a href="https://notebooks.antigenic-cartography.org/eu/notebook/?page_id=0708236">NIID incremental mege from 2007-NH VCM to 2007 SH TC2 (2007-09-01)</a>
  <li><a href="investigations/strain-selection-meeting/database/niid/investigations/overlay-merge/20070907-ref-ag-merge/incremental-merge-summary.html">Summary NIID incremental mege (2007-09-01)</a>

  <li><h3>NIMR</h3></a>

  <li><h3>Misc</h3></a>
  <li><a href="investigations/strain-selection-meeting/database/combined/investigations/avidity-test-comparisons/2007-SH-tc2-with-tc1/index.html">Avidity adjust comparisons TC1 and TC2</a>
  <li><a href="https://notebooks.antigenic-cartography.org/eu/notebook/?page_id=0709091">Avidity test on map used to start MELB TC1 to TC2 incremental merges</a>
  <p>
  <li><a href="https://notebooks.antigenic-cartography.org/eu/notebook/?page_id=0708271&lab=melb&images_index=0">NIID incremental mege from 2007-NH VCM to 2007 SH TC2 (2007-09-01)</a>
  <p>
  <li><a href="investigations/strain-selection-meeting/database/combined/investigations/sequence-name-matching/index.html">Sequence name matching</a>
  <li><a href="investigations/strain-selection-meeting/database/combined/investigations/sequence-name-matching/pulling-together-eus-processing.html">Pulling together Eu's sequence name matching (2007-09-16)</a>
  <p>
  <li><a href="https://notebooks.antigenic-cartography.org/eu/notebook/?page_id=0708271">Timeseries (2007-09-10)</a>
  <li><a href="https://notebooks.antigenic-cartography.org/eu/notebook/?page_id=070913">Timeseries (2007-09-14?)</a>
  <li><a href="https://notebooks.antigenic-cartography.org/eu/notebook/9000-0000-time-series/2007-0913-ts">Timeseries for ssm (2007-09-17)</a>
  <p>
  <li><a href="https://notebooks.antigenic-cartography.org/eu/eu-todo.html">Eu TODO (2007-09-13)</a>
  <p>
  <li><a href="investigations/strain-selection-meeting/database/combined/investigations/ag-effect-o-aa/2007-09-14/cdc/egg-cell-table/index.html">Regression on egg/cell table</a>
  <li><a href="investigations/strain-selection-meeting/database/combined/investigations/ag-effect-o-aa/2007-09-14/cdc/egg-cell-table-extracted-from-all-cdc/index.html">Regression on egg/cell table subset in CDC ssm map</a>
  <p>
  <li><a href="investigations/strain-selection-meeting/database/combined/investigations/ag-effect-o-aa/2007-09-14/cdc/full/index.html">CDC regression</a>
  <li><a href="investigations/strain-selection-meeting/database/combined/investigations/ag-effect-o-aa/2007-09-14/melb/full/index.html">MELB regression</a>
  <li><a href="investigations/strain-selection-meeting/database/combined/investigations/ag-effect-o-aa/2007-09-14/niid/full/index.html">NIID regression</a>
  <li><a href="investigations/strain-selection-meeting/database/combined/investigations/ag-effect-o-aa/2007-09-14/nimr/full/index.html">NIMR regression</a>
  </UL>


<H1> Strain-selection meeting 2008 NH -- misc investigations</H1>
  <UL>
  <li><a href="investigations/strain-selection-meeting/database/combined/investigations/sequence-name-matching/20071211-from-eu/pulling-together-eus-processing.html">Pulling together Eu's sequence name matching (2007-12-11)</a>
  <p>
  <li><a href="investigations/merge-hi-tables/4-lab-merge-nimr-pc-groups.html">NIMR PC into subsets (example) (2007-12-27)</a>
  <p>
  <li><a href="investigations/strain-selection-meeting/database/combined/investigations/2008-nh-misc/niid-serum-circles/index.html">NIID BR/10/2007 serum position (2008-01-06)</a>
  <p>
  <li><a href="investigations/strain-selection-meeting/database/combined/investigations/ag-effect-o-aa/2008-02-11/all/full/index-markings.html">Marking ag map with mutations</a>
  </UL>



<H1> Strain-selection meeting 2008 SH -- misc investigations</H1>
  <UL>
  <li><a href="investigations/turkey-gp/index.html">Comparison of turkey and gp RBCs on CDC experimental H3 HI table from 2008-06-18 (2008-07-28)</a>
  <li><a href="investigations/strain-selection-meeting/database/combined/investigations/2008-sh-misc/cdc-gp/index.html">Global (GUI) optizations on the master GP map (2008-08-14)</a>
  <p>
  <li><a href="investigations/strain-selection-meeting/database/combined/investigations/2008-sh-misc/melb-subset/index.html">Hand optimizations of MELB tables from 2007.5 run by themselves (2008-08-24)</a>
  <p>
  <li><a href="investigations/strain-selection-meeting/database/combined/investigations/2008-sh-misc/regressions/index.html">Ag effect of aa substs on SSM data (2007.5 to 2008.5 subsets, separate for each lab) (2008-08-24)</a>
  </UL>



<H1> Strain-selection meeting 2009 NH -- misc investigations</H1>
  <UL>
  <li><a href="investigations/strain-selection-meeting/database/combined/investigations/genetic-basis-of-turkey-rbc-problem/pca-regression.html">Genetic basis of turkey RBC avidity change--investigation with regression (2009-02-06)</a>
  <li><a href="investigations/strain-selection-meeting/database/combined/investigations/2009-nh-misc/ms-tables/index.html">H3 maps for strain selection meeting manuscript (2009-05-30)</a>
  </UL>


<H1> Strain-selection meeting 2009 SH -- misc investigations</H1>
  <UL>
  <li><a href="investigations/strain-selection-meeting/database/combined/investigations/2009-sh-misc/ts/H1N1pdm-CDC-NIID-NIMR/00_overview-with-labels.png.html">H1N1pdm timeseries (2009-09-19)</a>
  <li><a href="investigations/strain-selection-meeting/database/combined/investigations/2009-sh-misc/ts/H3/2009-01.a.png.html">H3 timeseries (2009-09-19)</a>
  <li><a href="investigations/strain-selection-meeting/database/combined/investigations/2009-sh-misc/ts/B-Victoria-with-geographic/0_overview.pdf.html">B Vic timeseries (2009-09-19)</a>
  </UL>


<H1> Retrospective CDC 1995-2002</H1>
  <UL>
  <li><a href="investigations/retrospective-cdc/index.html">Try global rearrangements on on the from-scratch map (2008-07-30)</a>
  </UL>



<H1> NIBSC strain selection for human pandemic library </H1>
  <UL>
  <li><a href="investigations/nibsc-library/h7/index.html">H7</a>
  </UL>

<H1> Spatial</H1>
  <UL>
  <li><a href="investigations/spatial/index.html">Working file (mostly for Derek)</a>
  <li><a href="investigations/spatial/index-locations.html">Locations on world map</a>
  <p>
  <li><a href="investigations/demo/index2.html">Web journal demo for Colin</a>
  <p>
  <li><h3>Geogrpahic and antigenic timeseries (End of May 2006)</h3></a>
  <li><a href="investigations/spatial/timeseries/geographic-antigenic-webpages-equirectangular-outline/small-2002-07.html">All WHO CCs. Geographic 2002-07 to 2005-12, and antigenic 2004-07 to 2005-12, on outline map</a>
  <li><a href="investigations/spatial/timeseries/geographic-antigenic-webpages-equirectangular-outline-cdc-amantadine/small-2002-07.html">All WHO CCs. Geographic 2002-07 to 2005-12, and CDC antigenic 2004-07 to 2005-12, on outline map also showing amantadine sensitivity</a>
  <li><a href="investigations/spatial/timeseries/geographic-antigenic-webpages-earth-satellite-same-color/small-2002-07.html">All WHO CCs. Geographic 2002-07 to 2005-12, and antigenic 2004-07 to 2005-12, on satellite map</a>
  <li><a href="investigations/spatial/timeseries/geographic-antigenic-webpages-earth-satellite/small-2002-07.html">All WHO CCs. Geographic 2002-07 to 2005-12, and antigenic 2004-07 to 2005-12, on satellite map with each CC in different color</a>
  <p>
  <li><h3>Geogrpahic and antigenic timeseries (Updated with some location information corrected June 8 2006)</h3></a>
  <li><a href="investigations/spatial/timeseries/geographic-antigenic-webpages-equirectangular-outline-update-1/small-2002-07.html">All WHO CCs. Geographic 2002-07 to 2005-12, and antigenic 2004-07 to 2005-12, on outline map (Update 1)</a>
  <li><a href="investigations/spatial/timeseries/geographic-antigenic-webpages-equirectangular-outline-cdc-amantadine-update-1/small-2002-07.html">All WHO CCs. Geographic 2002-07 to 2005-12, and CDC antigenic 2004-07 to 2005-12,, on outline map also showing amantadine sensitivity (Update 1)</a>
  <li><a href="investigations/spatial/timeseries/geographic-antigenic-webpages-earth-satellite-same-color-update-1/small-2002-07.html">All WHO CCs. Geographic 2002-07 to 2005-12, and antigenic 2004-07 to 2005-12, on satellite map (Update 1)</a>
  <li><a href="investigations/spatial/timeseries/geographic-antigenic-webpages-earth-satellite-update-1/small-2002-07.html">All WHO CCs. Geographic 2002-07 to 2005-12, and antigenic 2004-07 to 2005-12, on satellite map with each CC in different color (Update 1)</a>
  <p>
  <li><a href="investigations/spatial/timeseries/geographic-antigenic-webpages-equirectangular-outline-full-time-ag-maps/small-2002-07.html">All WHO CCs. Geographic and antigenic 2002-07 to 2005-12 on outline map</a>
  <li><a href="investigations/spatial/timeseries/geographic-antigenic-webpages-equirectangular-outline-full-time-ag-maps-20060908/small-2002-07.html">All WHO CCs. Geographic and antigenic 2002-07 to 2006-09 on outline map (2006 SH TC2)</a>
  <li><a href="investigations/spatial/timeseries/geographic-antigenic-webpages-equirectangular-outline-full-time-ag-maps-20060914/small-2002-07.html">All WHO CCs. Geographic and antigenic 2002-07 to 2006-08 on outline map (2006 SH consultation)</a>
  <li><a href="investigations/spatial/timeseries/geographic-antigenic-webpages-equirectangular-outline-full-time-ag-maps-20060914a/small-2002-07.html">All WHO CCs. Geographic and antigenic 2002-07 to 2006-08 on outline map (2006 SH consultation, different orientations for MELB and NIMR, mark different strains)</a>
  <p>
  <li><a href="investigations/spatial/hk-table.html">Look at HI table 20040512 from CDC, it has some HK strains in it, in a critical period for Colin's epidemiology</a>
  <p>
  <li><a href="investigations/spatial/sequences.html">Sequences</a>
  <p>
  <li><a href="investigations/spatial/state-league-table.html">US state league table</a>
  <p>
  <li><a href="investigations/strain-selection-meeting/database/melb/investigations/overlay-merge/20070209-grouped-names-ref-ag-merge/phil-seeding-oz.html">Phillipines seeing OZ (July 2003)</a>
  <p>
  <li><a href="investigations/spatial/table-comparisons.html">Compare CDC MELB and NIID tables pc's into 2006-09-14 composite map, full merge lab maps, and full-merge-overlay-merge maps (Colin and Derek and Eu) (2007 11 12)</a>
  </UL>


<H1> Mutagrip</H1>
  <UL>
  <li><a href="investigations/mutagrip/initial-map.html">Initial map</a>
  </UL>


<H1> Guus: CTL EC50 assay </H1>
  <UL>
  <li><a href="investigations/guus/index.html">CTL assay index (replicated in these following url's)</a>
  <li><a href="investigations/guus/ctl-assay/initial.html">CTL EC50 initial maps</a>
  <li><a href="investigations/guus/ctl-assay/corrected-threshold.html">CTL EC50 maps again, with corrected threshold</a>
  <li><a href="investigations/guus/ctl-assay/genetic-antigenic-comparison.html">Genetic/antigenic comparison</a>
  <li><a href="investigations/guus/ctl-assay/predictions.html">Looking more closely at ec50 predictions</a>
  </UL>

<H1> Guus misc </H1>
  <UL>
  <li><a href="investigations/guus/np-sequences/initial.html">Quick look at NP sequences</a>
  <p>
  <li><a href="investigations/guus/comutations/initial.html">Comutations</a>
  <p>
  <li><a href="investigations/guus/122av/alignment.html">122AV alignment</a>
  <li><a href="investigations/guus/122av/index.html">122AV investigations</a>
  </UL>

<H1> Glycan arrays </H1>
  <UL>
  <li><a href="investigations/glycan-arrays/with-miranda-200905/index.html">Initial poking around (with Miranda 2009-05-14)</a>
  <li><a href="investigations/glycan-arrays/with-miranda-200905/test-converting-data-to-binary.html">Test converting data to binary (with Miranda 2009-05-15)</a>
  <li><a href="investigations/glycan-arrays/with-miranda-200905/test-treating-data-as-linear.html">Test treating data as linear (with Miranda 2009-05-15)</a>
  <li><a href="investigations/glycan-arrays/with-miranda-200905/test-treating-data-as-linear-and-binary.html">Test treating data as linear and binary (with Miranda 2009-05-15)</a>
  <li><a href="investigations/glycan-arrays/with-miranda-200905/affinity.html">Initial exploration of affinity (with Miranda 2009-05-15)</a>
  <li><a href="investigations/glycan-arrays/larger-early-datasets.html">Larger early datasets (with Miranda 2009-05-27)</a>
  <li><a href="investigations/glycan-arrays/comparison-of-average-and-average-without-minmax.html">Comparison of using average or averge without min/max (with Miranda 2009-05-28)</a>
  <li><a href="investigations/glycan-arrays/2d-3d.html">2D 3D comparison (2009-06-01)</a>
  <li><a href="investigations/glycan-arrays/map-resolution.html">Map resolution (2009-06-01)</a>
  <li><a href="investigations/glycan-arrays/with-miranda-200905/thoughts.html">Thoughts / TODO (with Miranda 2009-05-15)</a>
  </UL>
<H1> Equine influenza AHT </H1>
  <UL>
  <li><a href="investigations/equine-aht/ferrets.html">First look at equine H3 with ferret data</a>
  <li><a href="investigations/equine-aht/sequences.html">First look at equine H3 sequences</a>
  <li><a href="investigations/equine-aht/shape-sequence.html">First look at equine H3 shape-sequence correspondence</a>
  <p>
  <li><a href="investigations/equine-aht/andrew.html">Ag distances for vaccine-challenge experiments</a>
  </UL>

<H1> Swine influenza</H1>
  <UL>
  <li><a href="investigations/swine/merge.html">Merge 5 HI tables</a>
  <li><a href="investigations/swine/merge-2.html">Follow up, just merge 3 HI tables</a>
  <li><a href="investigations/swine/tree-distances.html">Tree distances (for ron 2004-08-16)</a>
  <li><a href="investigations/swine/swine-map-with-human-science-ms-map.html">Merge with science ms human map (2004-08-16)</a>
  <li><a href="investigations/swine/jan-ron-ms-2006-04.html">Distances from PC/1/73 for manuscript (2006-04-16)</a>
  <li><a href="investigations/swine/hu-ag-rate.html">Rate of antigenic evolution in humans (for manuscript figure 2006-08-23)</a>
  <li><a href="investigations/swine/hu-genetic-rate.html">Rate of genetic evolution in humans (for manuscript figure 2006-09-26)</a>
  <li><a href="investigations/swine/slope-difference-testing.html">Slope difference testing (for manuscript figure 2006-10-01)</a>
  </UL>

<H1> Classic swine influenza</H1>
  <UL>
  <li><a href="investigations/swine-mexico/index.html">CDC 2009 04 23 HI table antigenic map</a>
  <li><a href="investigations/swine-mexico/20090430/index.html">CDC 2009 04 30 HI table antigenic map</a>
  <li><a href="investigations/swine-mexico/iowa-serum-mini-clusters/index.html">CDC 2009 06 03 HI table, explore the mini-cluster separation</a>
  <p>
  <li><a href="investigations/swine-mexico/regression/index.html">Regression (2009-06-23)</a>
  </UL>

<H1> HK swine influenza</H1>
  <UL>
  <li><a href="investigations/hk-swine/index.html">HK swine, sera investigations (2010-04-15)</a>
  <li><a href="investigations/hk-swine/take2/2000-map.html">HK swine, sera investigations.  Take 2.  2000 sera (2010-07-29)</a>
  <li><a href="investigations/hk-swine/take2/2004-map.html">HK swine, sera investigations.  Take 2.  2004 sera (2010-07-29)</a>
  <li><a href="investigations/hk-swine/take2/2009-map.html">HK swine, sera investigations.  Take 2.  2009 sera (2010-07-29)</a>
  <li><a href="investigations/hk-swine/take2/composite-map.html">HK swine, sera investigations.  Take 2.  2000 2004 and 2009 sera (2010-07-29)</a>
  <li><a href="investigations/hk-swine/take2/recapitulate-map-from-rapidd-meeting.html">Recapitulate map from RAPIDD meeting  2000 and 2009 (to April) sera (2010-07-29)</a>
  <li><a href="investigations/hk-swine/take2/4ags-all-sera.html">4 ags, all sera (2010-07-29)</a>
  </UL>



<H1> Immunocompromised child</H1>
  <UL>
  <li><a href="investigations/immunocompromised-child/index.html">Immunocompromised child (MELB)</a> at MELB only site <a href="https://notebooks.antigenic-cartography.org/dsmith/melb/immunocompromised-child/index.html">here</a>
  </UL>

<H1> NI</H1>
  <UL>
  <li><a href="investigations/ni/index.html">Initial look at NI data (2009-06-26) (CBER)</a>
  </UL>

<H1> Misc use</H1>
  <UL>
  <li><a href="investigations/1999-and-2000/1999-and-2000.html">Look at 1999 and 2000 tables</a>
  <li><a href="investigations/cr-assay/chromium-release-assay.html">Chromium release assay</a>
  <li><a href="investigations/cr-assay/chromium-release-assay-h1-and-h3-tables-combined.html">Chromium release assay (h1 and h3 combined)</a>
  <p>
  <li><a href="investigations/hiv/richman/initial.html">Richman initial maps</a>  
  <li><a href="investigations/hiv/richman/ferret.html">Ferret longitudinal</a>  
  <p>
  <li><a href="investigations/ron-h1-h16/main.html">H1 thru H16 seqences (ron)</a>
  <li><a href="investigations/misc/ron-na-similarity-matrices.html">HA and NA similarity matrices</a>
  <li><a href="investigations/misc/ron-na-similarity-genetic-maps.html">NA subtype genetic maps (INCORRECT, CORRECTED BELOW)</a>
  <li><a href="investigations/misc/ron-na-similarity-genetic-maps-take2.html">NA subtype genetic maps (take 2)</a>
  <li><a href="investigations/misc/na-phylip-output-for-ron.html">Phylip output of NA sequences, with gaps counted as one mutation</a>
  <p>
  <li><a href="investigations/misc/cosgrove-expansins.html">Expansins, Daniel Cosgrove, PSU</a>
  <p>
  <li><a href="investigations/immunocompromised-child/main.html">Immunocompromised child (MELB)</a>
  <p>
  <li><a href="investigations/lipman/index.html">NY strains (from PLOS paper)</a>
  <li><a href="investigations/eddie/nz-locations-for-eddie.html">NZ locations (for Eddie)</a>
  <li><a href="investigations/eddie/ny-state.html">NY locations (for Eddie)</a>
  <p>
  <li><a href="investigations/oli/index.html">Suentra/Marcos/Oli model data</a>
  <p>
  <li><a href="investigations/leptospirosis/index.html">Initial look at Jamie's lepto data</a>
  <li><a href="investigations/leptospirosis/index2.html">2nd look at Jamie's lepto data</a>
  <li><a href="investigations/leptospirosis/index3.html">3rd look at Jamie's lepto data -- add 4 other data sets</a>
  <p>
  <li><a href="investigations/scrub-typhus/index.html">Initial look at scrub typhus with Nick and Stewart at OUCRU Bangkok</a>
  <p>
  <li><a href="investigations/ev71/index.html">Initial look at EV71</a>
  <li><a href="investigations/ev71/2008-08/index.html">Second set of EV71 data (2008-08-08)</a>
  <li><a href="investigations/ev71/2008-08/index-updated.html">Second set of EV71 data, additional titers against B5s (2008-08-25)</a>
  <li><a href="investigations/ev71/2012-ag-ge/index.html">Genetic basis of antigenic change (2012-06-12 bJen-Ren & Judy)</a>
  <p>
  <li><a href="investigations/ibv/ibv.html">IBV initial analyses with David Brown (2010-07-20)</a>
  </UL>


<H1> Italy avaian library</H1>
  <UL>
  <li><a href="investigations/avian-library/h5/index.html">Initial tables</a>  
  <li><a href="investigations/avian-library/h5/index2.html">2nd set of tables -- RDE/Kolin comparison</a>  
  </UL>


<H1> HPV (with Mark Jit)</H1>
  <UL>
  <li><a href="investigations/hpv/index.html">Initial Combita data</a>  
  </UL>


<H1> Yoshi</H1>
  <UL>
  <li><a href="investigations/gates/index.html">Yoshi-specific index page (2011-03-22)</a>  
  <p>
  <li><a href="investigations/gates/h1-mutants/h1-mutants.html">H1 mutants (2011-03-22)</a>  
  <li><a href="investigations/gates/h1-mutants/regression.html">Regression (2011-03-28)</a>  
  <li><a href="investigations/gates/h1-mutants/update.html">H1 mutants update (2011-04-11)</a>  
  <li><a href="investigations/gates/h1-mutants/update-2011-07-06.html">H1 mutants update (2011-07-06)</a>  
  </UL>


<H1> HMPV DIs (with Bernadette)</H1>
  <UL>
  <li><a href="investigations/bernadette/index.html">Index for HMPV DI calcs below (supporting Bernadette 2012-12-29)</a>  
  <p>
  <li><a href="investigations/bernadette/initial-for-A.html">Initial exploration of Region A (2012-12-29)</a>  
  <li><a href="investigations/bernadette/initial-for-B.html">Initial exploration of Region B (2012-12-29)</a>  
  <p>
  <li><a href="investigations/bernadette/new-region-a.html">Redo region A (2013-01-05)</a>  
  <li><a href="investigations/bernadette/new-region-b.html">Redo region B (2013-01-12)</a>  
  <li><a href="investigations/bernadette/region-c.html">Region C, post 12k, end of genome (2013-01-26)</a>  
  <p>
  <li><a href="investigations/bernadette/num-muts.html">Num mutation calcs (2012-12-31) -- corrected in next link</a>  
  <li><a href="investigations/bernadette/num-muts-aG-tC-only.html">Num mutation calcs --redo (2013-02-01)</a>  
  <p>
  <li><a href="investigations/bernadette/expectation.html">Expectation of mutation (2012-12-30)</a>  
  <p>
  <li><a href="investigations/bernadette/motifs.html">Test whether the immediate neighboring bases affect edit probality (2013-01-13)</a>  
  <li><a href="investigations/bernadette/motifs-when-edited.html">Test whether the immediate neighboring bases affect edit probality REDO ABOVE CONDITIONAL ON READ EDITED (2013-02-09)</a>  
  <li><a href="investigations/bernadette/motifs-when-edited-including-region-c.html">Same as above, but including region C (2013-02-17)</a>  
  <li><a href="investigations/bernadette/motifs-when-edited-region-c-for-ms.html">Same as above, but for region C only, and done for ms1 (2013-07-01)</a>
  <p>
  <li><a href="investigations/bernadette/remove-duplicate-reads.html">Remove duplicate reads (2013-06-05)</a>  
  <li><a href="investigations/bernadette/num-muts-aG-tC-only-duplicate-reads-removed.html">Redo num mutations calcs with duplicate reads removed (2013-06-05)</a>  
  <p>
  <li><a href="investigations/bernadette/todo.html">Todo (2013-02-17)</a>  
  </UL>

<H1> Misc investigations</H1>
  <UL>
  <li><a href="investigations/misc/residuals-after-alan.html">Looking at target-fit residuals (after alan)</a>  
  <p>
  <li><a href="investigations/misc/bill-cluster-shapes.html">Do cluster shapes predict direction? (Bill Amos)</a>  
  <p>
  <li><a href="investigations/misc/walter-vaccine-efficacy-antigenic-distance.html">Walter, antigenic-distance correlation with vaccine efficacy</a>  
  <p>
  <li><a href="investigations/dengue/index.html">Initial look at Dengue, with the 'chine 'nual group (2012-07-19)</a>  
  </UL>


<H1> Misc supporting</H1>
  <UL>
  <li><a href="investigations/merge-hi-tables/reoriented.html">Reorienting coords for Alan</a>  
  <li><a href="investigations/merge-hi-tables/selecting-cluster-strains.html">Selecting cluster strains (supporting Ron 2004-08-17)</a>  
  <li><a href="investigations/ac-table-format/index.html">Antigenic Cartography table format</a>  
  <li><a href="investigations/misc/quasispecies-calcs.html">Quasispecies calcs (supporting Ron's inagural speech, 18 Dec 2007)</a>  
  </UL>



<H1> Data</H1>
  <UL>
  <li><a href="investigations/sequence-mds/passage-history-data.html">Passage history data for Dutch H3 (just what passaged in, not the history)</a>
  <li><a href="investigations/misc/collaborating-center-for-location.html">WHO CC for each location (for Ana to disambiguate)</a>
  </UL>


<H1> Demo</H1>
  <UL>
  <li><a href="investigations/demo-of-web-notebook/index.html">Demo of web notebook</a>  
  <li><a href="investigations/demo/sarah-sam-demo.html">Demo for Sarah and Sam (2010-08-05)</a>  
  <li><a href="investigations/demo/terry-and-barbara.html">Demo for Terry and Barbara (2013-07-31)</a>
  <li><a href="investigations/demo/david-demo.html">Demo for David (2014-12-03)</a>
  </UL>







<H1> Other</H1>
  <UL>
  <li>Power's study (repeated vaccination work, not mds related, I just wanted a link from this page) <a href="../../../..//projects/powers/index.html">local</a> <a href="http://notebooks.antigenic-cartography.org/dsmith/powers/index.html">at acorg</a>.
  <li>Analytic vaccination (repeated vaccination work, not mds related, I just wanted a link from this page) <a href="../projects/math-vac/index.html">local</a> <a href="http://www.santafe.edu/~dsmith/math-vac/index.html">at sfi</a>.
  <li>Equine (also linked above, but this to a place where AHT etc can go) <a href="investigations/equine-aht/andrew.html">local</a> <a href="http://notebooks.antigenic-cartography.org/dsmith/equine-aht/andrew.html">at ant cart</a>.
  <li>Rob HIV (nothing to do with MDS, just wanted a link from this page) <a href="../projects/hiv-rob/index.html">local</a> <a href="http://notebooks.antigenic-cartography.org/dsmith/hiv-rob/index.html">at ant cart</a>.
  <li>Mario/Sunetra/Oli model <a href="investigations/oli/index.html">local</a> <a href="http://notebooks.antigenic-cartography.org/dsmith/oli/index.html">at ant cart</a>.
  <li>Christophe's bacterial speciation model <a href="investigations/christophe/index.html">local</a> <a href="http://notebooks.antigenic-cartography.org/dsmith/christophe/index.html">at ant cart</a>.
  </UL>
<HR>


<CENTER><H1>User Interface</H1></CENTER>

<H1> Colors </H1>
  <UL>
  <li><a href="investigations/ui/colors/colors.html">Colors of the viral strains</a>
  <li><a href=https://notebooks.antigenic-cartography.org/eu/colors.html>X colors (Eu 2008-01-18)</a>
  </UL>

<H1> Misc tools </H1>
  <UL>
  <li><a href="investigations/ui/error-array-tool/error-array-tool.html">Error array tool</a>
  <li><a href="investigations/ui/point-stress-component-tool/main.html">Point stress component tool</a>
  <li><a href="investigations/procrustes-kmeans/first-run.html">Procrustes kmeans</a>
  <li><a href="investigations/ui/connection-tool/connection-tool.html">Connection tool</a>
  </UL>

<HR>

<CENTER><H1>Papers</H1></CENTER>

<H1>Method and validation</H1>
  <UL>
  <li><a href="investigations/validation/notes.html">Paper notes (Nov 2001)</a>
  <li><a href="investigations/validation/antigenic-map-notes.html">Antigenic Map paper notes (Dec 2001)</a>
  <li><a href="investigations/validation/genetic-map-notes.html">Genetic Map paper notes (Dec 2001)</a>
  <li><a href="investigations/validation/jan-prototypes-and-variants.html">Jan's prototypes and variants</a>
  <li><a href="investigations/validation/different-labs.html">Overlapping HI tables from different labs</a>
  <li><a href="investigations/validation/tabs123-removing-sr.html">Tables 1,2,3 merge, removing sera</a>
  </UL>


<H1>30 years of antigen and genetic evolution (comparison of sequence and shape)</H1>
  <UL>
  <li><a href="investigations/30-years-of-evolution/paper-notes.html">Paper notes</a>
  <li><a href="investigations/30-years-of-evolution/todo.html">To Do</a>
  <li><a href="papers/32-years/32Years.doc">Draft #2, 2003-05-14 (for discussion in Atlanta)</a>
  <li><a href="papers/32-years/32Years-draft3.doc">Draft #3, 2003-05-18 (for discussion in Atlanta, updated on plane)</a>
  <li><a href="papers/32-years/32Years-draft4.doc">Draft #4, 2003-05-28 (intermediate draft for Alan)</a>
  <li><a href="papers/32-years/32Years-draft5.doc">Draft #5, 2003-05-29 (intermediate draft for Ron)</a>
  </UL>

<H1>Evolution</H1>
  <UL>
  <li><a href="investigations/evolution/notes.html">Paper notes</a>
  </UL>

<H1>HI Prediction</H1>
  <UL>
  <li><a href="investigations/hi-prediction/paper-notes.html">Paper notes</a>
  </UL>

<H1>Ferret reinfection</H1>
  <UL>
  <li><a href="investigations/ferret-reinfection/manuscript-draft.html">Manuscript draft</a>
  </UL>


<H1> Antigenic carography of Influenza A(H5N1)</H1>
  <UL>
  <li><a href="investigations/h5/index.html">Antigenic map, CK/59 avidity</a>  
  <li><a href="investigations/h5/rates.html">Evolution rates</a>  
  <li><a href="investigations/h5/prediction.html">Predictions (to determine accuracy and dimension)</a>  
  <li><a href="investigations/h5/landscape-correlations.html">Landscape correlations</a>  (<a href="investigations/h5/landscape-correlations-test-outliers.html">Test outliers</a>)
  <li><a href="investigations/h5/figs1.html">Fig. S1, RBC comparisons</a>  
  <li><a href="investigations/h5/figs1-sera-comparison.html">Fig. S1, Sera comparisons</a>  
  <li><a href="investigations/h5/figs1-sera-comparison-adjust-for-slope.html">Fig. S1, Sera comparisons -- adjust for slope difference</a>  
  <li><a href="investigations/h5/joost.html">Fig. S5b redone for Joost ms (more animals)</a>  
  <p>
  <H3> Adding the Anuhi cluster (3 published strains) in the WER data for 2nd submission</H3>
  <li><a href="investigations/strain-selection-meeting/misc/h5/cdc/index-detailed.html">CDC data (18 aug 2006)</a>  
  <li><a href="investigations/strain-selection-meeting/misc/h5/cdc/index.html">CDC data (to cdc)</a>  
  <li><a href="investigations/strain-selection-meeting/misc/h5/cdc/wer.html">WER table (to cdc)</a>  
  <li><a href="investigations/h5/rates-with-anuhi.html">Evolution rates with Anuhi (same as <A href="investigations/h5/rates-with-anuhi.html">Evolution rates above, but with the addition of the Anuhi cluster</A>) </a>  
  <li><a href="investigations/strain-selection-meeting/misc/h5/cdc/regression.html">Antigenic efect of aa substitutions</a>  
  <li><a href="investigations/h5/figs1-sera-comparison-with-anuhi.html">Fig. 4, Sera comparisons (same as above, but with Anuhi)</a>  
  <li><a href="investigations/h5/figs1-with-anuhi.html">Fig. S1, RBC comparisons (with anuhi)</a>  
  <li><a href="investigations/h5/duck-ferret-rabbit-sera-with-anuhi.html">Fig. S1, Duck/Ferret/Rabbit sera comparisons (same as above, but with Anuhi)</a>  
  <p>
  <li><a href="investigations/strain-selection-meeting/misc/h5/cdc/2007-NH-VCM-25/index.html">Integrating 2 CDC tables from Feb 2007</a>  
  <p>
  <li><a href="investigations/strain-selection-meeting/misc/h5/combined/sequence/index.html">Processing the global H5 sequence data prior to tree building (2007-02-20)</a>  
  <p>
  <li><a href="investigations/h5/worldwide-sequences/index.html">Global H5 sequences, genetic maps of major clusters (2007-03-25)</a>  
  <p>
  <li><a href="investigations/h5/cluster-transition-mutations/index.html">The cluster transition substitutions for the ~2007-09 WHO map (2007-11-23)</a>  
  </UL>


<H1> Global epidemiology 2003-2006.5, persistence/seeding</H1>
  <UL>
  <li><a href="investigations/global-epidemiology/index.html">Outline and draft</a>  
  <li><a href="investigations/global-epidemiology/global-epi-ms-hacking/distances-to-trunk.html">Distributions of distances to trunk (2008-02-20)</a>  
  <li><a href="investigations/global-epidemiology/global-epi-ms-hacking/vodka-test-tree-construction.html">S-H tests for S America (2008-02-20)</a>  
  <p>
  <li><a href="investigations/global-epidemiology/global-epi-ms-hacking/global-epi-revision.html">Global epi revision (2008-02-23)</a>  
  </UL>





<hr>
<CENTER><H1>Presentations</H1></CENTER>
  <UL>
  <li><a href="presentations/2001-11-04-orthomyxovirus.ppt">2001-11-04 Orthomyxovirus</a>
  <li><a href="presentations/2001-11-28-curacao.ppt">2001-12-28 Curacao</a>
  <li><a href="presentations/2002-02-04-Princeton.ppt">2002-02-04 Princeton</a>
  <li><a href="presentations/2002-03-25-Nancy.ppt">2002-03-25 Nancy</a>
  <li><a href="investigations/jan-malta/initial.html">Jan Malta figure work</a>
  <li><a href="presentations/2002-10-23-Malta.ppt">2002-10-23 Malta</a>
  <li><a href="presentations/2003-05-03-Curacao.ppt">2003-05-03 Curacao</a>
  <li><a href="presentations/2003-05-18-CDC-Malta.ppt">2003-05-18 CDC Malta</a>
  <li><a href="presentations/2003-05-18-CDC-regression.ppt">2003-05-18 CDC regression</a>
  <li><a href="presentations/2003-05-18-CDC-seasons.ppt">2003-05-18 CDC seasons</a>
  <li><a href="presentations/2003-06-14-wildlife-ss.ppt">2003-06-14 Dutch wildlife summer school</a>
  <li><a href="presentations/2003-06-14-wildlife-ss-distributed.ppt">2003-06-14 Dutch wildlife summer school (distributed version)</a>
  <li><a href="presentations/2005-08-30-nih-d.ppt">2005-08-30 NIH Pioneer Award interview</a>
  <li><a href="presentations/2007-05-24-and-25-taiwan.ppt">2007-05-24-and-25 Taiwan (including updated intro to antigenic cartography and NICs acks slide)</a>
  <li><a href="presentations/2010-09-07-Options-VII-HK-u.ppt">2010-09-07 Options VII HK</a>
  <li><a href="presentations/2010-10-07-fitness-meeting-plaques.ppt">2010-10-07 Fitness meeting plaques</a>
  <li><a href="investigations/presentations/2014-12-05-EMC-masters.ppt">2014-12-05-EMC-masters</a>
  </UL>
<HR>


<CENTER><H1>Releases</H1></CENTER>

<H1>Research releases</H1>
  <UL>
  <li><a href="release/2001-01-03/mds.zip">2001-01-03</a>  Conjugant gradient optimization, and misc
  <li><a href="release/2001-10-11/mds.zip">2001-10-11</a>  HI Metric, error-lines, and misc
  <li><a href="release/2001-10-22/mds.zip">2001-10-22</a>  UI tweaks, including adding of printing coords
  <li><a href="release/2001-10-23/mds.zip">2001-10-23</a>  Fix error in 2001-10-22 printing of coords
  <li><a href="release/2001-10-24/mds.zip">2001-10-24</a>  Rulers (for measuring distances in the MDS plot)
  <li><a href="release/2001-10-25/mds.zip">2001-10-25</a>  Manual row and col adjusts.  Printing distances between points.
  <li><a href="release/2001-10-28/mds.zip">2001-10-28</a>  Disconnecting and reconnecting points.
  <li><a href="release/2001-10-29/mds.zip">2001-10-29</a>  Fix error when incomplete strain names.  Add grid lines.  Add fill-window.
  <li><a href="release/2001-11-06/mds.zip">2001-11-06</a>  Prediction.  Default col base max of highest titer in col and 1280
  <li><a href="release/2001-11-14/mds.zip">2001-11-14</a>  UI tweaks, modes, groups, rotate, zoom.  Fixed batch runs error.
  <li><a href="release/2001-11-18/mds.zip">2001-11-18</a>  Stop button, run indicator, names on and off, sera as rectangle, more prediction support.
  <li><a href="release/2001-11-19/mds.zip">2001-11-19</a>  Speed up prediction calculation
  <li><a href="release/2001-11-20/mds.zip">2001-11-20</a>  Fix new error in disconnecting points, fixed old error in error lines, make selection more like the mac, made disconnected points not be part of prediction.
  <li><a href="release/2001-11-21/mds.zip">2001-11-21</a>  Fixes/clean-up: Prediction, works nicely now with disconnect and reconnect; point-stress color coding now works and ignores disconnected points; new color for frozen points (dark grey).
  <li><a href="release/2001-11-23/mds.zip">2001-11-23</a>  Adds "Prediction lines", like error lines but showing prediction error.  Fixes old error in the initial state of the error-lines icon.
  <li><a href="release/2001-12-19/mds.zip">2001-12-19</a>  Adds saving and reading back configuration of plot window.  Adds saving distances and predictions to specified files.  Adds randomization of selected points.  
  <li><a href="release/2001-12-20/mds.zip">2001-12-20</a>  Adds saving and reading back configuration of batch window, and misc improvements to batch window.
  <li><a href="release/2002-01-05/mds.zip">2002-01-05</a>  Clean up of the table-window display, adding scroll bars for large tables, and pretty handling of HI tables.  Cosmetic adjustments in plot window.  Partially functioning scroll bars in plot window.  Stop button and new icon for batch runs.
  <li><a href="release/2002-01-06/mds.zip">2002-01-06</a>  Made point stress coloring zero based, not min point stress based.
  <li><a href="release/2002-01-09/mds.zip">2002-01-09</a>  Speed up outputting distances by removing fancy sorting.
  <li><a href="release/2002-01-11/mds.zip">2002-01-11</a>  Output distance matrix, reduce memory requirements for pairwise distance list.
  <li><a href="release/2002-01-14/mds.zip">2002-01-14</a>  Output reduced distance matrix, output table for selected points in mds plot.
  <li><a href="release/2002-01-31/mds.zip">2002-01-31</a>  Enhance the save tool so that it saves details such as colors and dot-sizes.  Add ability to have mds-plot names a different color than the strain dots.  Add 90 degree rotates, and 2x zooms (on shift-click).
  <li><a href="release/2002-02-04/mds.zip">2002-02-04</a>  Connection lines.  Prototype procrustes (unix only right now).
  <li><a href="release/2002-02-08/mds.zip">2002-02-08</a>  Input and output of lapedes-format.  Reduction in size of ag-sr-table save format (by 20x).
  <li><a href="release/2002-02-08a/mds.zip">2002-02-08a</a> Add Linux installation guide.  Add calling Xgobi from the mds plot (unix only).  Add debug for unix select problem.  Add command-line lapedes arguments.
  <li><a href="release/2002-02-09/mds.zip">2002-02-09</a>  Fix xgobi coords filename for some xgboi releases.
  <li><a href="release/2002-02-11/mds.zip">2002-02-11</a>  Work around tcl/tk point selection issue in some instalations.  Reduce size of large save files (by a further 3x).
  <li><a href="release/2002-02-12/mds.zip">2002-02-12</a>  Optimize disconnect/reconnect, freeze/unfreeze, and adjust so the mds-window is not updated for every point; instead the display update happens once for all points.
  <li><a href="release/2002-02-12a/mds.zip">2002-02-12a</a>  Zoom and rotate now work on just selected points if there are any selected, on all points otherwise.
  <li><a href="release/2002-02-14/mds.zip">2002-02-14</a>  Prototype method to specify details of the mds plot such as dot sizes and colors.  Pop-up windows for errors when reading hi tables or plot-spec files.
  <li><a href="release/2002-02-24/mds.zip">2002-02-24</a>  Prototype procrustes into Windows code.  Alan's dimensional annealing on Unix only.
  <li><a href="release/2002-03-04/mds.zip">2002-03-04</a>  Prototype dimensional annealing (for HI tables only) into Windows code.  Fix default loction where the UI looks for plot-specs.
  <li><a href="release/2002-03-14/mds.zip">2002-03-14</a>  Dim anneal log on windows.  New dim anneal schedules.  Dim anneal sees row and col adjusts from UI (hacked way).  Output of plot-specs in easily editable format.
  <li><a href="release/2002-03-19/mds.zip">2002-03-19</a>  Try to fix the previous dim anneal release for windows.  Unify output commands from a plot window onto a single menu.
  <li><a href="release/2002-03-20/mds.zip">2002-03-20</a>  Add automatic row and col adjusts of selected strains.
  <li><a href="release/2002-06-13/mds.zip">2002-06-13</a>  Error and connection lines now only show for selected points instead of all points.  The default plot-spec can now be a fuction as well as a value.  Procrustes option to scale or not to scale.  10x speedup of the lisp optimizer.  Added arbitrary, and mixed, threshold values.  for now just <10 and <40 are there, but we can have any.  Save format now includes the graphics details such as the size of the window the save was made from an the rotation/scale/tranlation of the data in the window.  New mds windows made from existing windows (2nd icon on the left) are made the same size as the exisiting window.  The rotation/scale/translation of the data in a window is specific for that window.
  <li><a href="release/2002-06-13a/mds.zip">2002-06-13a</a>  Fix error, latest mds-window size not stored in the save format.
  <li><a href="release/2002-07-10/mds.zip">2002-07-10</a>  Add polygon, and polygon-scaled shapes (including in the plot spec, which is currently the only way to set circle, rectangle, polygon and polygon-scaled).  Also, 100x speedup for launching the map window for large (1000ish strains) tables.
  <li><a href="release/2002-08-08/mds.zip">2002-08-08</a>  Integrate Alan's Procrustes kmeans clusterer into the GUI (unix only, not done for windows yet).
  <li><a href="release/2002-09-20/mds.zip">2002-09-20</a>  Allow <20 threshold.
  <li><a href="release/2002-09-26/mds.zip">2002-09-26</a>  Initial CTL assay integration.  OK for now, but known deficiencies are fixed homolgous at 7 (or max in col), titers displayed in the table window are junk (the software thinks it an HI table for display purposes), the table must contain at least one >5000 or >10000 to be identified as a CTL-assay table, and only >5000 and >10000 "greater than" titers are allowed.  Each of these is not too hard to fix, this is an alpha release to get started.
  <li><a href="release/2002-10-02/mds.zip">2002-10-02</a>  Dynamically assigned thresholds, specification of similarity matrices.
  <li><a href="release/2002-10-06/mds.zip">2002-10-06</a>  Procrustes for MS Windows.  (just procrustes, not the procrustes-kmeans clusterer yet).
  <li><a href="release/2002-10-07/mds.zip">2002-10-07</a>  Match names when procrustesing HI (with -ag and -sr suffixes) and sequence (wihtout the suffixes).
  <li><a href="release/2002-10-13/mds.zip">2002-                rror (missing functionality) whereby rows and cols autoadjust worked when running interactively, but not in batch mode from the GUI.
  <li><a href="release/2002-10-15/mds.zip">2002-10-15</a>  Add code to generate jo-wu prediction tables.
  <li><a href="release/2002-11-01/mds.zip">2002-11-01</a>  Fixed error lines for threshold values (was off by one, causing non-balanced looking error lines).  Fixed hi table printing from mds-window to write to file.  Fixed hi table printing to write * not . so the table is re-readable.  Fixed hi table printing to figure width of antigen names and print accordingly.  Added option to color points with _average_ stress (previously we only had absolute stress).  Allow list of titers for non-averaged merges, and bootstrapping titers.
  <li><a href="release/2002-11-21/mds.zip">2002-11-21</a>  Split up the visualize-mds-coords function, to get around the heap size limit in acl 6.2 on windows.
  <li><a href="release/2003-01-26/mds.zip">2003-01-26</a>  Add setting of dot colors from the GUI.
  <li><a href="release/2003-01-28/mds.zip">2003-01-28</a>  Fix windows error, Allegro 6.2 vs 4.2 depreciation.
  <li><a href="release/2003-02-02/mds.zip">2003-02-02</a>  Add Lispworks compatibility.
  <li><a href="release/2003-11-27/mds.zip">2003-11-27 source</a>  and windows binary: <a href="release/2003-11-27/mds-x.exe"> tcl version 8.3 </a> and <a href="release/2003-11-27/mds.exe">tcl version independent</a>  Fixed procruestes in lispworks on windows.
  <li><a href="release/2004-07-27/mds.zip">2004-07-27</a>  Release for CVS.
  <li><a href="release/2004-08-07/mds.zip">2004-08-07</a>  Minor changes to batch.lisp and save.lisp for scripting.
  <li><a href="release/2004-08-13/mds.zip">2004-08-13</a>  Initial cut at table report script.
  <li><a href="release/2004-08-26/mds.zip">2004-08-26</a>  Back out the clinit.cl.rest hack as we no longer use openmcl.  Update the installation-notes for windows.  Add tree reordering, and tree distance matrix printing.  Add merge hi tables (not done merge into an existing map yet).  Added name size and color, and dot size and color and shape, menu items to the map window (icon name is "plot spec").
  <li><a href="release/2004-08-29/mds.zip">2004-08-29</a>  Try fix from Franz for the XP lock up of the GUI (did not work)
  <li><a href="release/2004-09-01/mds.zip">2004-09-01</a>  Try an idea from Terry, to call tclsh instead of wish, for the XP lock up of the GUI.
  <li><a href="release/2004-09-03/mds.zip">2004-09-03</a>  Add tree renaming (top level GUI function).
  <li><a href="release/2004-09-06/mds.zip">2004-09-06</a>  Add tree distance list printing (top level GUI function).
  <li><a href="release/2004-09-06a/mds.zip">2004-09-06a</a>  Sort strain names in tree distance list printing, add overlay merge saves into top-level GUI, add reading in a treefile in the top-level GUI, remove gratituous warning re no-standard-strain-abbreviation.
  <li><a href="release/2004-09-08/mds.zip">2004-09-08</a>  Add removing of duplicates from fasta files to the top level gui.
  <li><a href="release/2004-11-05/mds.zip">2004-11-05</a>  Fix bit rot in parsing R regression output (for moving to R version 1.9.1 from 1.7)
  <li><a href="release/2004-11-19/mds.zip">2004-11-19</a>  Update INSTALLATION-NOTES and test new CVS-based release mechanism
  <li><a href="release/2004-11-29/mds.zip">2004-11-29</a>  Fix list-to-string utility which broke going from acl6.2 to 7.0 (error manifested in failed procrustes call due to filename not being correct)
  <li><a href="release/2004-11-29a/mds.zip">2004-11-29a</a>  Fix error in release script which was not copying the gridware dim-anneal code, which caused the user's system load to fail just before starting the input-ui window
  <li><a href="release/2004-12-28/mds.zip">2004-12-28</a>  Debitrot gridware submission and cmucl lisp code at sfi after tree update (latest code now running at sfi, old code in mds.old), includes udpates to utils.system, mds.system, make-mds-release, clinit.cl for sfi
  <li><a href="release/2005-07-26/mds.zip">2005-07-26</a>  Add a dead-sparrow button to the input-ui to hack around windows lockup in opening and saving files
  <li><a href="release/2005-08-11/mds.zip">2005-08-11</a>  Add separate dead-sparrow window for lockups from the input-ui.  Remove F33 from the ferret list (not used, and interferes with the concentric circle test case).  Add running dim-anneal once to the batch ui.
  <li><a href="release/2005-08-18/mds.zip">2005-08-18</a>  Incorporate Ron's new positions for the dead-sparrow and input-ui windows so the Allego GUI on windows does not cover them.  Add option to the save menu in a map window to write in pymol format for 3D viewing in pymol.  (Name the outputu files <something>.pml and from the pymol gui do file->run and then the filename.)
  <li><a href="release/2005-08-19/mds.zip">2005-08-19</a>  Fix error in setting default shape.  Unclear why this error has not been seen before.  
  <li><a href="release/2005-10-06/mds.zip">2005-10-06</a>  New pymol format for grid, bounding box, notches, etc. (Terry).  Output distances from root strain, through cluster centroids (Derek).
  <li><a href="release/2005-10-09/mds.zip">2005-10-09</a>  3D Procrustes in pymol.  Procrustes of lower dimensional map onto higher dimensional map.  GUI file selection filters to limit files to specific types (by default, any time available). Add pymol plugin to the release. (Terry and Derek.)
  <li><a href="release/2005-10-15/mds.zip">2005-10-15</a>  Add antigenic effect amino acid substitutions calculations to the GUI. (and internally bring the antigenic effect of substitutions regression code from investigations code into the main-line distribution) (Derek).  Updated pymol antigenic map plugin (Derek thinks it fixes the awkwardness when first toggeling the display of map components) (Terry).
  <li><a href="release/2005-12-01/mds.zip">2005-12-01</a>  Add procrustes of selected points only.
  <li><a href="release/2006-04-03/mds.zip">2006-04-03</a>  Add reading directly from fasta file into MDS.
  <li><a href="release/2006-04-03/mds.zip">2006-04-12</a>  AHT/Malaria bleeding edge clean up (details in email to be added here)
  <li><a href="release/2006-04-16/mds.zip">2006-04-16</a>  Use "*" instead of dont-care in the output from "Distances from table" and "Distances from table and map" options in the save menu of the map window.
  <li><a href="release/2006-04-17/mds.zip">2006-04-17</a>  Launch pymol 3d viewer from the map-window.
  <li>Newer releases are now <A href="http://notebooks.antigenic-cartography.org/dsmith/mds/release/index.html">here</A> at antigenic-cartography.org (and <A href="/home/dsmith/mds/src/mds/mds/release/index.html">here</A> locally)
  </UL>




